{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "import PyPDF2\n",
    "from dotenv import load_dotenv\n",
    "import openai\n",
    "import pandas as pd\n",
    "from sklearn.metrics.pairwise import cosine_similarity\n",
    "import json\n",
    "from langchain.text_splitter import CharacterTextSplitter\n",
    "import sentence_transformers\n",
    "from langchain.embeddings import OpenAIEmbeddings, HuggingFaceInstructEmbeddings\n",
    "from langchain.vectorstores import Chroma, FAISS\n",
    "from langchain.chat_models import ChatOpenAI\n",
    "from langchain.prompts import PromptTemplate\n",
    "from langchain.memory import ConversationBufferMemory\n",
    "from langchain.chains import ConversationalRetrievalChain, LLMChain\n",
    "from langchain.llms import HuggingFaceHub\n",
    "from langchain.document_loaders import DirectoryLoader, PyPDFDirectoryLoader, CSVLoader\n",
    "from langchain.indexes import VectorstoreIndexCreator\n",
    "import getpass\n",
    "from IPython.display import display, HTML"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total papers:  476\n",
      "Current index:  101\n",
      "Current index:  202\n",
      "Current index:  303\n",
      "Current index:  404\n",
      "Current index:  505\n"
     ]
    }
   ],
   "source": [
    "def search_semantic_scholar(query,year,n=100):\n",
    "    current_index = 0\n",
    "    r = requests.get(\n",
    "    'https://api.semanticscholar.org/graph/v1/paper/search?query='+query+'&year='+year+'-',\n",
    "    params={'fields': 'title,year,abstract,url,openAccessPdf,tldr,isOpenAccess,publicationDate'}\n",
    "    )\n",
    "    #print(json.dumps(r.json(), indent=2))\n",
    "    r = r.json()\n",
    "    #total number of papers found \n",
    "    total = r['total']\n",
    "    offset = r['offset']\n",
    "    #nextindex = r['next']\n",
    "    final_results = pd.json_normalize(r['data'])\n",
    "    current_index = offset+n+1\n",
    "    print('Total papers: ',total)\n",
    "    print('Current index: ',current_index)\n",
    "    while current_index < total:\n",
    "        r = requests.get(\n",
    "        'https://api.semanticscholar.org/graph/v1/paper/search?query='+query+'&offset='+str(current_index)+'&year='+year+'-'+'&limit='+str(n),\n",
    "        params={'fields': 'title,year,abstract,url,openAccessPdf,tldr,isOpenAccess,publicationDate'}\n",
    "        )\n",
    "        #print(json.dumps(r.json(), indent=2))\n",
    "        r = r.json()\n",
    "        offset = r['offset']\n",
    "        #nextindex = r['next']\n",
    "        results = pd.json_normalize(r['data'])\n",
    "        final_results = pd.concat([final_results, results], ignore_index=True)\n",
    "        current_index = offset+n+1\n",
    "        print('Current index: ',current_index)\n",
    "    return final_results\n",
    "\n",
    "\n",
    "instance = search_semantic_scholar(query=\"HER3 ErBb3\",year=\"2022\",n=100)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>paperId</th>\n",
       "      <th>url</th>\n",
       "      <th>title</th>\n",
       "      <th>abstract</th>\n",
       "      <th>year</th>\n",
       "      <th>isOpenAccess</th>\n",
       "      <th>publicationDate</th>\n",
       "      <th>openAccessPdf.url</th>\n",
       "      <th>openAccessPdf.status</th>\n",
       "      <th>tldr.model</th>\n",
       "      <th>tldr.text</th>\n",
       "      <th>openAccessPdf</th>\n",
       "      <th>tldr</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>a2adc9d9eb46537935bbaa40d38703d43391defa</td>\n",
       "      <td>https://www.semanticscholar.org/paper/a2adc9d9...</td>\n",
       "      <td>Stromal-derived NRG1 enables oncogenic KRAS by...</td>\n",
       "      <td>Activating KRAS mutations (KRAS*) in pancreati...</td>\n",
       "      <td>2023</td>\n",
       "      <td>False</td>\n",
       "      <td>2023-09-29</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>tldr@v2.0.0</td>\n",
       "      <td>It is found that CAFs can contribute toKRAS* i...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>a2adc9d9eb46537935bbaa40d38703d43391defa</td>\n",
       "      <td>https://www.semanticscholar.org/paper/a2adc9d9...</td>\n",
       "      <td>Stromal-derived NRG1 enables oncogenic KRAS by...</td>\n",
       "      <td>Activating KRAS mutations (KRAS*) in pancreati...</td>\n",
       "      <td>2023</td>\n",
       "      <td>False</td>\n",
       "      <td>2023-09-29</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>tldr@v2.0.0</td>\n",
       "      <td>It is found that CAFs can contribute toKRAS* i...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>da8a72d58dfeaee4b019fa63499f03bff54396a6</td>\n",
       "      <td>https://www.semanticscholar.org/paper/da8a72d5...</td>\n",
       "      <td>A multi-ancestry genome-wide association study...</td>\n",
       "      <td>Type 1 diabetes is a chronic autoimmune diseas...</td>\n",
       "      <td>2023</td>\n",
       "      <td>True</td>\n",
       "      <td>2023-09-18</td>\n",
       "      <td>https://www.medrxiv.org/content/medrxiv/early/...</td>\n",
       "      <td>GREEN</td>\n",
       "      <td>tldr@v2.0.0</td>\n",
       "      <td>A multi-ancestry GWAS enabled identification o...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>679151a019a3ac96b4937863a42d0d05934019f5</td>\n",
       "      <td>https://www.semanticscholar.org/paper/679151a0...</td>\n",
       "      <td>Role of human epidermal growth factor receptor...</td>\n",
       "      <td>ABSTRACT Background ALK tyrosine kinase inhibi...</td>\n",
       "      <td>2023</td>\n",
       "      <td>True</td>\n",
       "      <td>2023-09-18</td>\n",
       "      <td>https://www.tandfonline.com/doi/pdf/10.1080/15...</td>\n",
       "      <td>GOLD</td>\n",
       "      <td>tldr@v2.0.0</td>\n",
       "      <td>Evidence is provided that HER3 may mediate TKI...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>df947244810339cfddf7a075c1059e49aca19724</td>\n",
       "      <td>https://www.semanticscholar.org/paper/df947244...</td>\n",
       "      <td>HER3 functions as an effective therapeutic tar...</td>\n",
       "      <td>None</td>\n",
       "      <td>2023</td>\n",
       "      <td>True</td>\n",
       "      <td>2023-09-16</td>\n",
       "      <td>https://cancerci.biomedcentral.com/counter/pdf...</td>\n",
       "      <td>GOLD</td>\n",
       "      <td>tldr@v2.0.0</td>\n",
       "      <td>The data demonstrate that increased HER3 is an...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>255</th>\n",
       "      <td>0e8f9d5527245866bfc68962b2e4934392f4380f</td>\n",
       "      <td>https://www.semanticscholar.org/paper/0e8f9d55...</td>\n",
       "      <td>Upstream Process Design of Antibody-drug Conju...</td>\n",
       "      <td>: Colorectal cancer is a common tumor in the d...</td>\n",
       "      <td>2022</td>\n",
       "      <td>True</td>\n",
       "      <td>None</td>\n",
       "      <td>https://francis-press.com/uploads/papers/ca5w2...</td>\n",
       "      <td>BRONZE</td>\n",
       "      <td>tldr@v2.0.0</td>\n",
       "      <td>The antibody selected by the drug is patrituma...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>a1dd42028996f9414c4d6677d1f3fc0ea5e09650</td>\n",
       "      <td>https://www.semanticscholar.org/paper/a1dd4202...</td>\n",
       "      <td>Enabling Precision Therapy in Oncology: Liquid...</td>\n",
       "      <td>MYD88 PIK3C2G RAD51B SMAD4 TP53 ACVR1 BRCA1 CR...</td>\n",
       "      <td>2022</td>\n",
       "      <td>False</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>tldr@v2.0.0</td>\n",
       "      <td>PIK3C2G RAD54L SNCAIP TSC2 AKT3 BTG2 CSF3R ERC...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>61a6ec6b7130b84f024c3f6508be4b76a6a43220</td>\n",
       "      <td>https://www.semanticscholar.org/paper/61a6ec6b...</td>\n",
       "      <td>TITLE: Dasatinib Resensitizes MAPK Inhibitor E...</td>\n",
       "      <td>Resistance to combination BRAF/MEK inhibitor (...</td>\n",
       "      <td>2023</td>\n",
       "      <td>False</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>tldr@v2.0.0</td>\n",
       "      <td>It is proposed that dasatinib-based MAPKi ther...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>48871c227f1da93fe6b2656f9772cd941c920f7c</td>\n",
       "      <td>https://www.semanticscholar.org/paper/48871c22...</td>\n",
       "      <td>A Microfluidic SERS Assay to Characterize the ...</td>\n",
       "      <td>None</td>\n",
       "      <td>2023</td>\n",
       "      <td>False</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>tldr@v2.0.0</td>\n",
       "      <td>ESCP is demonstrated for characterizing the ex...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>361</th>\n",
       "      <td>2b3679f989cf1fc301eb9149fd76cfb8f136fd36</td>\n",
       "      <td>https://www.semanticscholar.org/paper/2b3679f9...</td>\n",
       "      <td>TITLE: HER2 mediates PSMA/mGluR1-driven resist...</td>\n",
       "      <td>45 Prostate cancer remains a major cause of ma...</td>\n",
       "      <td>2022</td>\n",
       "      <td>False</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>tldr@v2.0.0</td>\n",
       "      <td>The results suggest a novel therapeutic possib...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>382 rows × 13 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                      paperId  \\\n",
       "192  a2adc9d9eb46537935bbaa40d38703d43391defa   \n",
       "210  a2adc9d9eb46537935bbaa40d38703d43391defa   \n",
       "312  da8a72d58dfeaee4b019fa63499f03bff54396a6   \n",
       "218  679151a019a3ac96b4937863a42d0d05934019f5   \n",
       "79   df947244810339cfddf7a075c1059e49aca19724   \n",
       "..                                        ...   \n",
       "255  0e8f9d5527245866bfc68962b2e4934392f4380f   \n",
       "275  a1dd42028996f9414c4d6677d1f3fc0ea5e09650   \n",
       "310  61a6ec6b7130b84f024c3f6508be4b76a6a43220   \n",
       "322  48871c227f1da93fe6b2656f9772cd941c920f7c   \n",
       "361  2b3679f989cf1fc301eb9149fd76cfb8f136fd36   \n",
       "\n",
       "                                                   url  \\\n",
       "192  https://www.semanticscholar.org/paper/a2adc9d9...   \n",
       "210  https://www.semanticscholar.org/paper/a2adc9d9...   \n",
       "312  https://www.semanticscholar.org/paper/da8a72d5...   \n",
       "218  https://www.semanticscholar.org/paper/679151a0...   \n",
       "79   https://www.semanticscholar.org/paper/df947244...   \n",
       "..                                                 ...   \n",
       "255  https://www.semanticscholar.org/paper/0e8f9d55...   \n",
       "275  https://www.semanticscholar.org/paper/a1dd4202...   \n",
       "310  https://www.semanticscholar.org/paper/61a6ec6b...   \n",
       "322  https://www.semanticscholar.org/paper/48871c22...   \n",
       "361  https://www.semanticscholar.org/paper/2b3679f9...   \n",
       "\n",
       "                                                 title  \\\n",
       "192  Stromal-derived NRG1 enables oncogenic KRAS by...   \n",
       "210  Stromal-derived NRG1 enables oncogenic KRAS by...   \n",
       "312  A multi-ancestry genome-wide association study...   \n",
       "218  Role of human epidermal growth factor receptor...   \n",
       "79   HER3 functions as an effective therapeutic tar...   \n",
       "..                                                 ...   \n",
       "255  Upstream Process Design of Antibody-drug Conju...   \n",
       "275  Enabling Precision Therapy in Oncology: Liquid...   \n",
       "310  TITLE: Dasatinib Resensitizes MAPK Inhibitor E...   \n",
       "322  A Microfluidic SERS Assay to Characterize the ...   \n",
       "361  TITLE: HER2 mediates PSMA/mGluR1-driven resist...   \n",
       "\n",
       "                                              abstract  year  isOpenAccess  \\\n",
       "192  Activating KRAS mutations (KRAS*) in pancreati...  2023         False   \n",
       "210  Activating KRAS mutations (KRAS*) in pancreati...  2023         False   \n",
       "312  Type 1 diabetes is a chronic autoimmune diseas...  2023          True   \n",
       "218  ABSTRACT Background ALK tyrosine kinase inhibi...  2023          True   \n",
       "79                                                None  2023          True   \n",
       "..                                                 ...   ...           ...   \n",
       "255  : Colorectal cancer is a common tumor in the d...  2022          True   \n",
       "275  MYD88 PIK3C2G RAD51B SMAD4 TP53 ACVR1 BRCA1 CR...  2022         False   \n",
       "310  Resistance to combination BRAF/MEK inhibitor (...  2023         False   \n",
       "322                                               None  2023         False   \n",
       "361  45 Prostate cancer remains a major cause of ma...  2022         False   \n",
       "\n",
       "    publicationDate                                  openAccessPdf.url  \\\n",
       "192      2023-09-29                                                NaN   \n",
       "210      2023-09-29                                                NaN   \n",
       "312      2023-09-18  https://www.medrxiv.org/content/medrxiv/early/...   \n",
       "218      2023-09-18  https://www.tandfonline.com/doi/pdf/10.1080/15...   \n",
       "79       2023-09-16  https://cancerci.biomedcentral.com/counter/pdf...   \n",
       "..              ...                                                ...   \n",
       "255            None  https://francis-press.com/uploads/papers/ca5w2...   \n",
       "275            None                                                NaN   \n",
       "310            None                                                NaN   \n",
       "322            None                                                NaN   \n",
       "361            None                                                NaN   \n",
       "\n",
       "    openAccessPdf.status   tldr.model  \\\n",
       "192                  NaN  tldr@v2.0.0   \n",
       "210                  NaN  tldr@v2.0.0   \n",
       "312                GREEN  tldr@v2.0.0   \n",
       "218                 GOLD  tldr@v2.0.0   \n",
       "79                  GOLD  tldr@v2.0.0   \n",
       "..                   ...          ...   \n",
       "255               BRONZE  tldr@v2.0.0   \n",
       "275                  NaN  tldr@v2.0.0   \n",
       "310                  NaN  tldr@v2.0.0   \n",
       "322                  NaN  tldr@v2.0.0   \n",
       "361                  NaN  tldr@v2.0.0   \n",
       "\n",
       "                                             tldr.text  openAccessPdf  tldr  \n",
       "192  It is found that CAFs can contribute toKRAS* i...            NaN   NaN  \n",
       "210  It is found that CAFs can contribute toKRAS* i...            NaN   NaN  \n",
       "312  A multi-ancestry GWAS enabled identification o...            NaN   NaN  \n",
       "218  Evidence is provided that HER3 may mediate TKI...            NaN   NaN  \n",
       "79   The data demonstrate that increased HER3 is an...            NaN   NaN  \n",
       "..                                                 ...            ...   ...  \n",
       "255  The antibody selected by the drug is patrituma...            NaN   NaN  \n",
       "275  PIK3C2G RAD54L SNCAIP TSC2 AKT3 BTG2 CSF3R ERC...            NaN   NaN  \n",
       "310  It is proposed that dasatinib-based MAPKi ther...            NaN   NaN  \n",
       "322  ESCP is demonstrated for characterizing the ex...            NaN   NaN  \n",
       "361  The results suggest a novel therapeutic possib...            NaN   NaN  \n",
       "\n",
       "[382 rows x 13 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "display(instance.sort_values('publicationDate',ascending=False))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "loader = PyPDFDirectoryLoader(\".\", glob=\"**/[!.]*.pdf\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/gid00030/gid00035/gid00032/gid00030/gid00038/gid00001/gid00033/gid00042/gid00045 /gid00001\\n/gid00048/gid00043/gid00031/gid00028/gid00047/gid00032/gid00046Citation: Miano, C.; Morselli, A.;\\nPontis, F.; Bongiovanni, C.; Sacchi, F.;\\nDa Pra, S.; Romaniello, D.; Tassinari,\\nR.; Sgarzi, M.; Pantano, E.; et al.\\nNRG1/ERBB3/ERBB2 Axis Triggers\\nAnchorage-Independent Growth of\\nBasal-like/Triple-Negative Breast\\nCancer Cells. Cancers 2022 ,14, 1603.\\nhttps://doi.org/10.3390/\\ncancers14071603\\nAcademic Editors: Reiner Strick and\\nRamona Erber\\nReceived: 4 February 2022\\nAccepted: 16 March 2022\\nPublished: 22 March 2022\\nPublisher’s Note: MDPI stays neutral\\nwith regard to jurisdictional claims in\\npublished maps and institutional afﬁl-\\niations.\\nCopyright: © 2022 by the authors.\\nLicensee MDPI, Basel, Switzerland.\\nThis article is an open access article\\ndistributed under the terms and\\nconditions of the Creative Commons\\nAttribution (CC BY) license (https://\\ncreativecommons.org/licenses/by/\\n4.0/).\\ncancers\\nArticle\\nNRG1/ERBB3/ERBB2 Axis Triggers Anchorage-Independent\\nGrowth of Basal-like/Triple-Negative Breast Cancer Cells\\nCarmen Miano1,2, Alessandra Morselli3, Francesca Pontis4\\n, Chiara Bongiovanni1,2,3, Francesca Sacchi1,2,3\\n,\\nSilvia Da Pra1,2,3, Donatella Romaniello3\\n, Riccardo Tassinari1, Michela Sgarzi3, Elvira Pantano4,\\nCarlo Ventura1,3, Mattia Lauriola3,†\\nand Gabriele D’Uva1,2,3,*,†\\n1National Laboratory of Molecular Biology and Stem Cell Engineering, National Institute of Biostructures and\\nBiosystems (INBB), 40129 Bologna, Italy\\n2Centre for Applied Biomedical Research (CRBA), University of Bologna, 40138 Bologna, Italy\\n3Department of Experimental, Diagnostic and Specialty Medicine, University of Bologna, 40138 Bologna, Italy\\n4Scientiﬁc and Technological Pole, IRCCS MultiMedica, 20138 Milan, Italy\\n*Correspondence: gabrielematteo.duva2@unibo.it\\n† These authors contributed equally to this work.\\nSimple Summary: Breast cancer is a heterogeneous disease, categorized into distinct subgroups with\\ndifferent clinical prognoses and treatment strategies. This study aimed to evaluate the role of ERBB3\\nin different molecular subtypes of breast cancers. Despite ERBB3/HER3 and its partner ERBB2/HER2\\nshowing low expression levels in basal-like/triple-negative breast cancers, stratiﬁcation of basal-like\\npatients according to ERBB3 mRNA expression levels highlighted a correlation between higher ERBB3\\nlevels and shorter relapse-free patients’ survival. In vitro analyses unveiled that the activation of\\nthe NRG1/ERBB3/ERBB2 axis robustly induces anchorage-independent growth of basal-like/triple-\\nnegative breast cancer cellular models, without signiﬁcant effects on cell proliferation, differentiation,\\nand migration in adhesion. Overall, our data suggest that ERBB2/ERBB3 plays an oncogenic role\\nin basal-like/triple-negative breast cancer patients, suggesting its neutralization as a therapeutic\\nstrategy for these breast cancer subtypes, which today have very limited treatment opportunities.\\nAbstract: ERBB3, also known as HER3, is a tyrosine kinase transmembrane receptor of the ERBB\\nfamily. Upon binding to neuregulin 1 (NRG1), ERBB3 preferentially dimerizes with HER2 (ERBB2),\\nin turn inducing aggressive features in several cancer types. The analysis of a dataset of breast\\ncancer patients unveiled that higher ERBB3 mRNA expression correlates with shorter relapse-free\\nsurvival in basal-like breast cancers, despite low ERBB3 expression in this breast cancer subtype.\\nAdministration of neuregulin 1 beta (NRG1 β) signiﬁcantly affected neither cellular proliferation nor\\nthe basal migratory ability of basal-like/triple-negative quasi-normal MCF10A breast cells, cultured\\nin mono-layer conditions. Furthermore, no signiﬁcant regulation in cell morphology or in the\\nexpression of basal/myoepithelial and luminal markers was observed upon stimulation with NRG1 β.\\nIn non-adherent conditions, NRG1 βadministration to MCF10A cells did not signiﬁcantly inﬂuence\\ncell survival; however, it robustly induced cell growth as spheroids (3D growth). Intriguingly, a\\nremarkable upregulation of ERBB3 and ERBB2 protein abundance was observed in 3D compared\\nto 2D cell cultures, and NRG1 β-induced 3D cell growth was efﬁciently prevented by the anti-\\nHER2 monoclonal antibody pertuzumab. Similar results were obtained by the analysis of basal-\\nlike/triple-negative breast cancer cellular models, MDA-MB-468 and MDA-MB-231 cells, in which\\nNRG1 βinduced anchorage-independent cell growth that in turn was prevented or reduced by the\\nsimultaneous administration of anti-HER2 neutralizing antibodies. Finally, the ability of pertuzumab\\nin suppressing NRG1 β-induced 3D growth was also evaluated and conﬁrmed in MCF10A engineered\\nwith HER2-overexpression. We suggest that the NRG1/ERBB3/ERBB2 pathway promotes the\\nanchorage-independent growth of basal-like breast cancer cells. Importantly, we provide evidence\\nthat ERBB2 neutralization, in particular by pertuzumab, robustly inhibits this process. Our results\\npave the way towards the development of novel anticancer strategies for basal-like breast cancer\\npatients based on the interception of the NRG1/ERBB3/ERBB2 signaling axis.\\nCancers 2022 ,14, 1603. https://doi.org/10.3390/cancers14071603 https://www.mdpi.com/journal/cancersCancers 2022 ,14, 1603 2 of 19\\nKeywords: basal-like breast cancer; triple-negative breast cancer; neuregulin 1; ERBB3; HER3;\\nERBB2; HER2; anchorage-independent growth; 3D growth; spheroids; HER2-targeting agents;\\npertuzumab; trastuzumab\\n1. Introduction\\nERBB/HER receptors (EGFR/ERBB1, ERBB2/HER2, ERBB3/HER3, and ERBB4/HER4)\\nplay major roles in the development and progression of various solid cancers, including\\nthose arising from the breast tissue (reviewed in [ 1–3]). Upon binding of speciﬁc ligands,\\nERBB receptors form homo- and/or hetero-dimers with other family members, and activate\\nmultiple downstream signaling cascades (reviewed in [ 1–3]). Neuregulins (NRGs) are\\ngrowth factors binding to ERBB3 and/or ERBB4, predominantly inducing heterotypic\\ninteractions with ERBB2 (reviewed in [ 1–4]). ERBB3 is involved in tumor development\\nand progression of different cancer types, including breast cancer (reviewed in [ 1,5]).\\nIn addition, ERBB3 mutations are not uncommon in many cancer types [ 6] (reviewed\\nin [7]). ERBB2 is unable to bind ligands, yet it is the preferred hetero-dimerization partner\\nfor the other ERBBs, amplifying and diversifying the signaling cascades [ 8] (reviewed\\nin [1–3]). ERBB3 and ERBB2 levels are frequently associated, especially in human breast\\ncancers, where ERBB3/ERBB2 heterodimers function as an oncogenic driver for breast\\ntumor cell proliferation [ 9] (reviewed in [ 1,5]). ERBB3 is detectable in preneoplastic ERBB2-\\noverexpressing breast epithelium and is required for cell growth and malignant progression\\nin ERBB2-driven hyperplasia [ 10]. The NRG1-ERBB3-ERBB2 signaling axis has been shown\\nto induce the self-renewal and proliferation of breast tumor-initiating cells and cancer stem\\ncells [11,12].\\nBreast cancer is a heterogeneous disease. For this reason, it is classiﬁed into different\\nsubtypes with distinct clinical behaviors and prognoses and different treatment strategies\\n(reviewed in [ 13,14]). Therefore, the accurate classiﬁcation of breast cancer subtypes is\\ncrucial for personalized disease management and for improving patient outcomes. In\\nclinical practice, breast cancer classiﬁcation driving the treatment-decision process is es-\\ntablished on the distinction between major breast cancer subsets based on conventional\\nimmunohistochemistry (IHC) and in situ hybridization (ISH) analyses of ERBB2 (HER2),\\nestrogen receptor alpha (ER), and progesterone receptor (PR) (reviewed in [ 13,14]). Based\\non IHC/ISH analyses, three clinical breast cancer subgroups can be identiﬁed: HER2+,\\nhormone receptor-positive, and triple-negative. Breast cancer expressing ERBB2/HER2\\nare classiﬁed as HER2+, independently from the expression of ER and/or PR. Conversely,\\nbreast cancers expressing ER and/or PR, but not ERBB2/HER2, are termed “hormone\\nreceptor-positive”, “hormone responsive” or “luminal” tumors. Tumors not expressing\\nER, PR, and ERBB2/HER2 are called “triple-negative” (TNBC). Anti-HER2 agents are\\nrecommended for the treatment of HER2+ breast cancer patients, whereas the admin-\\nistration of ER antagonists is recommended to hormone receptor-positive breast cancer\\npatients. Unfortunately, less efﬁcient targeted treatment strategies are currently available\\nfor “triple-negative” cancer patients. Indeed, the only targeted therapy currently approved\\nfor triple-negative breast cancers is pembrolizumab (an antibody against programmed\\ndeath-receptor PD-1 that provides dual ligand blockade of programmed death-ligands PD-\\nL1 and PD-L2). However, about 1 out of 5 triple-negative breast cancer patients expresses\\nPD-L1 protein, thus only a small subset of triple-negative patients can beneﬁt from the\\ncombinatorial administration of pembrolizumab and chemotherapy (reviewed in [14,15]).\\nIn the last two decades, different breast cancer molecular subtypes have been identiﬁed\\nby gene expression data [ 16–18] (reviewed in [ 13,14,19]). Molecular subtypes of breast\\ncarcinomas include basal-like (BLBC), HER2-enriched, luminal A, and luminal B. In this\\nregard, basal-like breast cancers are characterized by poor prognosis and a high rate\\nof relapse [ 20–22] (reviewed in [ 23–25]). Notably, basal-like and triple-negative breast\\ncancers exhibit a high degree of gene expression proﬁle overlap, with ∼80% of BLBCsCancers 2022 ,14, 1603 3 of 19\\nbeing ER-negative and HER2-negative, and 50–75% of TNBCs displaying a basal-like\\nphenotype [ 20–22] (reviewed in [ 23–25]). Unfortunately, similar to triple-negative breast\\ncancers, basal-like cancer subtypes have very limited treatment opportunities to date.\\nIn this study, we investigated the potential role of NRG1/ERBB3/ERBB2 signaling in\\ndifferent clinical and molecular subtypes of breast cancers, by combining metanalysis on\\npatients’ survival and gene expression, and by exploring in vitro potential effects induced\\nby the activation of this pathway.\\n2. Materials and Methods\\n2.1. Bioinformatic Analysis of Breast Cancer Patients’ Data\\nEvaluation of mRNA expression of genes of interest in normal breast tissue and/or\\nbreast cancers stratiﬁed for molecular and clinical subtypes was conducted by “bc-\\nGenExMiner” [ 26] (http://bcgenex.ico.unicancer.fr/, accessed on 9 March 2022) and “UAL-\\nCAN” [ 27] (http://ualcan.path.uab.edu/, accessed on 9 March 2022), respectively. Eval-\\nuation of mRNA expression of genes of interest in breast cancer cell lines stratiﬁed for\\nmolecular and clinical subtypes was conducted by “Gene expression-based Outcome for\\nBreast cancer Online” (GOBO) [ 28] (http://co.bmc.lu.se/gobo/gsa.pl, accessed on 9 March\\n2022). GOBO gene set expression analysis in breast cancer cell lines (GSA-Cell line) includes\\nmRNA expression data across 51 breast cancer cell lines [29].\\nAnalyses of relapse-free survival (RFS) of breast cancer patients were conducted by the\\nKaplan-Meier (KM) plotter online database [ 30] (http://kmplot.com/, accessed on 9 March\\n2022). KM plotter sources for the databases include GEO, EGA, and TCGA. Patients\\nbelonging to speciﬁc breast cancer molecular subtypes were stratiﬁed into two groups\\naccording to their expression levels of ERBB3 mRNA, and relapse-free survival (RFS) after\\ntumor resection was calculated by the Kaplan–Meier curve and log-rank test during a\\nfollow-up period of 120 months. The best performing threshold between the lower and\\nupper quartiles was used as a cutoff. The results were shown in the Kaplan–Meier survival\\nplots. p-value, hazard ratio (HR), and false discovery rate (FDR) are provided. p-value\\n< 0.05 by using the log-rank test was regarded as statistically signiﬁcant. Hazard ratios\\nwere used to estimate the effect for time-to-event end points, such as relapse-free survival.\\nA hazard ratio of 1 means lack of association, a hazard ratio greater than 1 suggests an\\nincreased risk, and a hazard ratio below 1 suggests a smaller risk.\\n2.2. Cell Cultures\\nIn vitro experiments have been conducted in breast cell line MCF10A and in pre-\\nviously generated HER2-overexpressing MCF10A (MCF10A-HER2) and their control\\n(MCF10A-ctrl) cells [ 31,32]. MCF10A cells and derivative lines (MCF10A-HER2 and\\nMCF10A-ctrl) were kindly provided by Prof. Yarden’s laboratory (Weizmann Institute of\\nScience) and were cultured in DMEM-F12 medium containing 5% of horse serum (HS),\\n1% penicillin/streptomycin, 1% L-glutamine, 0.5 µg/mL of hydrocortisone, 100 ng/mL\\nof cholera toxin, 0.5 µg/mL of insulin and 10 ng/mL of EGF (hereafter “full medium”).\\nFurther experiments have been performed on breast cancer cell lines MDA-MB-468 and\\nMDA-MB-231. Both the cell lines were cultured in RPMI-1640 medium containing 10% fetal\\nbovine serum (FBS). The cells have been grown in 10 cm plastic Petri dishes and incubated\\nat 37◦C in a humidiﬁed atmosphere of 5% CO 2/air.\\n2.3. Proliferation and Random/Directional Migration Analysis in Monolayer Conditions\\nProliferation and migration analyses on MCF10A cells were performed using a\\nLivecyteTMtechnology (Phase Focus, Shefﬁeld, UK). Cells were treated with neuregulin 1 β\\n(10 ng/mL), trastuzumab (10 µg/mL), and pertuzumab (10 µg/mL), alone and in combina-\\ntion, before the start of the experiment. Trastuzumab or pertuzumab was added at least\\n30 min before adding NRG1 β. Images were acquired every 60 min for 72 h with a 10 ×ob-\\njective, at 37◦C and 5% CO 2/air. Data were analyzed using Cell Analysis Toolbox software\\n(Phase Focus, Shefﬁeld, UK). Cell proliferation was determined by Livecyte software byCancers 2022 ,14, 1603 4 of 19\\ncounting the number of cells in each frame. Cell motility was evaluated by measuring cell\\nvelocity, calculated as the change in position in each frame. The degree of directional versus\\nrandom migration was estimated by calculating cell displacement and the cell conﬁnement\\nratio. Indeed, these two parameters represent the distance a cell migrates relative to its\\npoint of origin and consider the degree to which a cell meanders from its starting and end\\npoints. In particular, cell displacement shows the position of cells and their trajectories over\\ntime, relative to their point of origin. Conﬁnement ratio is the ratio of the length of the direct\\npath between the initial and the ﬁnal position over the total track length. To circumvent the\\nproblem of dependency on the cell track duration, the conﬁnement ratio was multiplied by\\nthe square root of time. Morphological and morphometrical analyses were performed by\\nLivecyte software calculating the area of cells in each frame, the sphericity, measuring how\\nclose to a sphere is a cell in each frame, and the length to width ratio, deriving from the\\ncalculation of how round versus elongated a cell is in each frame.\\nThen, 50,000 MDA-MB-468 and MDA-MB-231 cells/well were seeded into a six-well\\nplate in full medium. Treatments with neuregulin 1 β(10 ng/mL) were added the day after\\nseeding. After three days of treatments, cells were trypsinized and manually counted using\\nthe Neubauer Chamber. Representative pictures were acquired by using EVOS ™M5000\\nImaging System at 20 ×magniﬁcation.\\n2.4. Anchorage-Independent Growth Assay\\nTo assess the ability of cells to grow in anchorage-independent conditions, a growth\\nassay was carried out in ultra-low attachment six-well plates. For each well, 10,000 cells\\nwere seeded in 2 mL of complete medium. Treatments with neuregulin 1 β, trastuzumab,\\nand pertuzumab, alone and in combination, were added immediately and repeated every\\nthree days. After two weeks of treatment, pictures of the wells were collected using an\\ninverted microscope at 4 ×magniﬁcation. Spheroids from each condition were counted\\nand areas were measured using ImageJ software.\\n2.5. RNA Isolation and Real Time PCR\\nNext, 100,000 MCF10A cells were seeded in six-well plates in full medium. Treatments\\nwere added and repeated every 3–4 days. After 9 days, cultured cells were washed,\\ntrypsinized, and collected. Total RNA was extracted with a NucleoSpin RNA II kit\\n(Macherey Nagel), following the manufacturer’s protocol speciﬁcations. Total RNA quan-\\ntity and quality were determined using a NanoDrop spectrophotometer (N1000, Thermo\\nFisher Scientiﬁc, Waltham, MA, USA). RNA reverse transcription to double-stranded\\ncDNA was performed using SuperScript ™VILO ™cDNA Synthesis Kit (Invitrogen ™,\\nWaltham, MA, USA), according to the manufacturer’s protocol, followed by incubation in\\nthe thermocycler for the reaction. Real Time (rt)-PCR was performed with a QuantStudio\\n6 Flex instrument (Applied Biosystems, Waltham, MA, USA) by using Fast SYBR Green\\nPCR Master Mix (Applied Biosystems, Waltham, MA, USA), and analysis was conducted\\non QuantStudio 6 Flex Real-Time PCR System software. Oligonucleotide sequences of\\nanalyzed genes, namely KRT5, KRT14, TP63, KRT8, MUC1, and HPRT1 (Sigma-Aldrich,\\nSaint Louis, MO, USA), are listed in Supplementary Table S1. Relative quantiﬁcation was\\nperformed using HPRT1 gene as a loading control. DDCT was calculated and data of each\\ngene were analyzed using a 2−DDCTmethod and reported as mean fold change.\\n2.6. Analysis of Cell Cycle Activity on Cultured Cells by Immunoﬂuorescence Evaluation of KI67\\nThen, 2000 MCF10A cells/well were seeded into a 96-well plate in full medium.\\nTreatments with neuregulin 1 β, trastuzumab, and pertuzumab, alone and in combination,\\nwere added immediately and repeated every 3 days. After 9 days of treatments, cells were\\nﬁxed in 4% paraformaldehyde (PFA) for 20 min. Samples were washed and permeabilized\\nwith 0.5% Triton-X100 (Sigma-Aldrich, Saint Louis, MO, USA) in PBS for 5 min at room\\ntemperature. Then, to avoid the non-speciﬁc binding of the antibodies, samples were\\nblocked with PBS supplemented with 5% bovine serum albumin (BSA) (Sigma-Aldrich,Cancers 2022 ,14, 1603 5 of 19\\nSaint Louis, MO, USA) and 0.1% Triton-X100 for 1 h at room temperature. To analyze cell-\\ncycle re-entry, samples were incubated overnight at 4◦C with anti-KI67 primary antibody\\n(1:100, ab16667, Abcam, Cambridge, UK) diluted in PBS supplemented with 3% BSA and\\n0.1% Triton-X100. After primary antibody incubation, samples were washed 3 times in\\nPBS and incubated at room temperature for 1 h with the ﬂuorescent secondary antibody\\nanti-rabbit 594 (1:200, AlexaFluor 111-585-003, Jackson), diluted in PBS supplemented with\\n1% BSA and 0.1% Triton-X100. Afterward, samples were washed three times in PBS. Then,\\nfor nuclei visualization, samples were stained for 15 min at room temperature with DAPI\\n(4’,6-diamidino-2-phenylindole dihydrochloride, Sigma-Aldrich, Saint Louis, MO, USA),\\ndiluted in PBS (1:1000). Samples were then washed twice more in PBS. Finally, pictures\\nwere acquired by using a Zeiss wideﬁeld microscope (Axio Observer A1) or an ArrayScan\\nXTI wideﬁeld microscope (Thermo Fisher Scientiﬁc, Waltham, MA, USA). The number of\\nKI-67 positive cells was quantiﬁed using ImageJ software.\\n2.7. Protein Extraction and Evaluation by Western Blot Analysis\\nMCF10A cells were seeded on 6 cm plastic Petri dishes (500,000 cells) and cultured as a\\nmonolayer. After 4 days of culture, they were washed, trypsinized, and collected. MCF10A\\ncells grown in anchorage-independent conditions (10,000 cells/well) were washed and\\ncollected after 14 days of culture. Both adherent and non-adherent MCF10A cells were\\nresuspended in RIPA buffer supplemented with a protease inhibitor cocktail (P8340, Sigma-\\nAldrich, Saint Louis, MO, USA, 1:100) and Na 3VO 4(1 mM). Then, 65 µg of protein extracts\\nwere resolved by sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis and\\ntransferred to a nitrocellulose membrane (AmershamTM ProtranTM Premium 0.45 µm\\n300 mm ×4 m). Next, the membrane was blocked for 60 min using TBS-T (0.1% Tween-20)\\nsupplemented by 3% BSA (Sigma-Aldrich, Saint Louis, MO, USA), and incubated overnight\\n(4◦C) with the following primary antibodies: anti-ERBB2 monoclonal antibody (1:1000\\ndilution; #4290 Cell Signaling Technology, Inc., Danvers, MA, USA), anti-ERBB3 (1:1000\\ndilution; #4754 Cell Signaling Technology, Inc., Danvers, MA, USA), anti-GAPDH (1:1000\\ndilution; #G9545 Sigma-Aldrich, Saint Louis, MO, USA). For protein detection, the mem-\\nbrane was incubated with anti-rabbit horseradish peroxidase-labeled secondary antibody\\n(Dako EnVision+ System- HRP Labelled Polymer) followed by a chemiluminescent reaction\\n(Clarity Western ECL Substrate, Bio-Rad). Signals and images were acquired by Chemi\\nDoc™XRS 2015 (Bio-Rad Laboratories, Hercules, CA, USA), and densitometric analysis\\nwas performed using Image Lab software (version 5.2.1; Bio-Rad Laboratories, Hercules,\\nCA, USA). The original western blot ﬁgures can be found in Figure S9.\\n2.8. Cell Survival Analysis by Flow Cytometry\\nMCF10A cells were seeded on ultra-low attachment 6-well plates. For each well,\\n100,000 cells were seeded in 2 mL of full medium. Treatments with neuregulin 1 βwere\\nadded immediately. After 24, 48, and 72 h, respectively, cells were harvested and washed\\ntwice in PBS. Annexin V-APC was added to the cell suspension for 15 min, according to the\\nmanufacturer’s protocol (Biolegend, San Diego, CA, USA). After Annexin V staining, cells\\nwere resuspended in Annexin binding buffer and propidium iodide (PI) was added. Cells\\nwere analyzed by CytoFLEX Flow cytometer through CytExpert software. Cells positive\\nfor Annexin V-APC were identiﬁed as early apoptotic cells, whereas cells positive for both\\nAnnexin V-APC and propidium iodide were considered as late apoptotic/necrotic cells. A\\nminimum of 10,000 events were recorded for each sample.\\n2.9. Statistical Analysis\\nStatistical analyses were performed with GraphPad Prism 8 software. Whenever\\nnormality could be assumed, statistical differences between group means were determined\\nusing the two-sided Student’s t-test or analysis of variance (ANOVA) followed by Tukey’s\\nor Sidak’s test, as speciﬁed in the ﬁgure legends. p-value < 0.05 was considered to represent\\na statistically signiﬁcant difference. In all panels, numerical data are expressed as meanCancers 2022 ,14, 1603 6 of 19\\n±standard error of the mean (s.e.m.); results are marked with one asterisk (*) if p< 0.05,\\ntwo (**) if p< 0.01, three (***) if p< 0.001, and four (****) if p< 0.0001. The sample size was\\ndetermined considering the variability observed in preliminary and similar experiments.\\n3. Results\\n3.1. mRNA Expression Levels of ERBB3 Correlate with Shorter Relapse-Free Survival in Basal-like\\nBreast Cancer Patients\\nWe started our investigation by evaluating the correlation between ERBB3 abundance\\nand the relapse-free survival (RFS), namely the time that the patient survives without any\\ncancer sign after primary treatment. To this end, breast cancer patients were divided into\\ndifferent molecular subtypes and stratiﬁed into two groups according to the expression\\nlevels of ERBB3 mRNA. Breast cancer patients’ stratiﬁcation in molecular subtype was\\nperformed by a prediction analysis of microarray 50 (PAM50) [ 33], a 50-gene signature\\nthat classiﬁes breast cancer into ﬁve subgroups: basal-like, HER2-enriched, luminal A,\\nluminal B, and normal-like. RFS after tumor resection was calculated during a follow-\\nup period of 10 years. Surprisingly, we found that higher ERBB3 mRNA levels predict\\na signiﬁcantly shorter RFS in basal-like breast cancer patients (Figure 1a), whereas no\\nsigniﬁcant correlation was observed in other breast cancer subtypes (Figure 1b–d). The\\ncorrelation between increased ERBB3 mRNA levels and lower RFS was quite substantial.\\nIndeed, the group of patients with higher ERBB3 mRNA levels compared with lower ERBB3\\nmRNA levels reported an 80% increased risk (hazard ratio = 1.8). These data suggest an\\noncogenic role of ERBB3 in basal-like breast cancer patients.\\nWe continued our investigation by evaluating the mRNA expression levels of the\\nERBB3 receptor in different molecular subtypes of breast cancer patients. Our data show\\nthat the average expression levels of ERBB3 mRNA modestly diverge in different breast\\nmolecular subtypes (Figure 2a). In detail, luminal A and B breast cancer patients displayed\\nthe highest ERBB3 mRNA levels, HER2-enriched and normal-like breast cancer patients\\nshowed intermediate levels and basal-like breast cancer patients exhibited the lowest\\nlevels. Similarly, the evaluation of the mRNA expression levels of ERBB3 in breast cancer\\nspecimens stratiﬁed for clinical subtypes demonstrated that the luminal, HER2+, and\\ntriple-negative subgroup display the highest, intermediate and lowest levels, respectively\\n(Figure 2b). Furthermore, the analysis of the ERBB3 ligand NRG1 unveiled very low levels\\nin breast cancer tissue compared to normal specimens, with minimal differences among\\nclinical and molecular breast cancer subtypes (Supplementary Figure S1a,b). These data\\nsuggest that ERBB3 exerts an oncogenic role in basal-like/triple-negative tumors, despite\\nthe low expression levels of ERBB3 receptor in these breast cancer subtypes.Cancers 2022 ,14, 1603 7 of 19\\nCancers 2022 , 14, x 7 of 20 \\n \\n suggest that ERBB3 exerts an oncogenic role in basal-like/triple-negative tumors, despite \\nthe low expression levels of ERBB3 receptor in these breast cancer subtypes. \\n \\nFigure 1. Relapse-free survival of breast cancer patients stratified for molecular subtypes and ERBB3 \\nmRNA levels. ( a–d) Relapse-free survival (RFS) of ( a) basal-like (n = 437), ( b) HER2-enriched (n = \\n352), ( c) luminal A (n = 624), and ( d) luminal B (n = 559) breast cancer patients stratified into two \\ngroups according to ERBB3  mRNA levels. Stratification in molecular subtypes was obtained by a \\nprediction analysis of microarray  50 (PAM50). Data on normal-lik e subgroup are not provided due \\nto a low number of patients. p-value, hazard ratio (HR), and false discovery rate (FDR) are also \\nprovided. \\nFigure 1. Relapse-free survival of breast cancer patients stratiﬁed for molecular subtypes and ERBB3\\nmRNA levels. ( a–d) Relapse-free survival (RFS) of ( a) basal-like ( n= 437), ( b) HER2-enriched ( n= 352 ),\\n(c) luminal A ( n= 624), and ( d) luminal B ( n= 559) breast cancer patients stratiﬁed into two groups\\naccording to ERBB3 mRNA levels. Stratiﬁcation in molecular subtypes was obtained by a prediction\\nanalysis of microarray 50 (PAM50). Data on normal-like subgroup are not provided due to a low\\nnumber of patients. p-value, hazard ratio (HR), and false discovery rate (FDR) are also provided.\\nCancers 2022 , 14, x 8 of 20 \\n \\n  \\nFigure 2. Basal-like/triple-negative br east cancers express low ERBB3  mRNA levels compared to \\nother breast cancer subtypes.  (a) Box plot of ERBB3  gene expression in breast cancer patients strati-\\nfied for molecular subtypes (P AM50, n = 10,065 patients); ( b) box plot of ERBB3  gene expression in \\nbreast cancer patients stratified for clinical subtypes (n = 719 patients); p-values are provided; * p < \\n0.05 and **** p < 0.0001. \\n3.2. Neuregulin 1 β (NRG1β) Does Not Significantly Induce Proliferation or Motility in Basal-\\nlike/Triple-Negative Breast Cells Cu ltured in Monolayer Conditions \\nBasal-like breast cancer is an aggressive molecular subtype characterized by a molec-\\nular signature similar to that of basal myoe pithelial cells, and nowadays the treatment \\noptions are very few. To evaluate a potential role for ERBB3 in basal-like breast cancer cell aggressiveness, we decided to perform in vitr o experiments upon activation of this recep-\\ntor. To select an appropriate cellul ar model, we analyzed the levels of ERBB3  mRNA in 51 \\nbreast cell lines, which were previously char acterized and divided into luminal and basal-\\nlike (basal A and basal B) subgroups [29]. Basal-like cell lines, in particular within the basal \\nB subgroup, showed lower ERBB3  mRNA levels compared to luminal cell lines (Supple-\\nmentary Figure S2a,b), mirroring the observation in basal-like breast cancer patients. In \\naccordance, triple-negative breast cancer exhibits low expression levels of ERBB3  com-\\npared to the other clinical subtypes (Suppl ementary Figure S2c). Furthermore, a modest \\nincrease in NRG1  mRNA levels was observed in the basal-B subgroup (Supplementary \\nFigure S2d,e), while the stratification for clinical breast cancer subtypes highlighted a high variability in the triple-negative subgroup without significant differences with the other \\nsubtypes (Supplementary Figu re S2f). Among basal-like cell lines, we then selected \\nMCF10A cells as the cellular model. These cells were derived from benign breast tissue \\nspontaneously immortalized without defined factors [34]. Thus, the employment of \\nMCF10A cells as a cellular model avoids the interference of multiple aberrations usually occurring in cancer cells. In addition, MCF10A cells are a widely employed and accepted \\nmodel to study the mechanistic contributi on of different ERBB family members \\n[31,32,35,36]. MCF10A cells, as most basal B cell lines, express very low levels of ERBB2 (Supplementary Figure S3a–c),  ER-alpha (Supplementary Fi gure S3d–f), and PR (Supple-\\nmentary Figure S3g–i), thus they are also co nsidered “triple-negative” [37] (reviewed in \\n[38]). \\nTo evaluate a potential role for ERBB3 signaling in basal-like/triple-negative breast \\ncells, we employed the ERBB3 ligand neuregulin 1 β (NRG1β). Firstly, MCF10A cells were \\ntreated with medium containing NRG1 β (10 ng/mL) or with control medium for seven \\ndays, then fixed and analyzed for cell cycle acti vity by immunostaining of KI67 protein. \\nIndeed, nuclear KI67 immunoreactivity is detec table during all the active phases of the \\ncell cycle (G1, S, G2, and M phases), but not in resting cells (G0 phase). However, our \\nresults did not unveil significant differences in the percentage of cells in the active phases \\nFigure 2. Basal-like/triple-negative breast cancers express low ERBB3 mRNA levels compared to\\nother breast cancer subtypes. ( a) Box plot of ERBB3 gene expression in breast cancer patients stratiﬁed\\nfor molecular subtypes (PAM50, n= 10,065 patients); ( b) box plot of ERBB3 gene expression in breast\\ncancer patients stratiﬁed for clinical subtypes ( n= 719 patients); p-values are provided; * p< 0.05,\\n**p< 0.01, *** p< 0.001, and **** p< 0.0001.Cancers 2022 ,14, 1603 8 of 19\\n3.2. Neuregulin 1 β(NRG1 β) Does Not Signiﬁcantly Induce Proliferation or Motility in\\nBasal-like/Triple-Negative Breast Cells Cultured in Monolayer Conditions\\nBasal-like breast cancer is an aggressive molecular subtype characterized by a molec-\\nular signature similar to that of basal myoepithelial cells, and nowadays the treatment\\noptions are very few. To evaluate a potential role for ERBB3 in basal-like breast cancer\\ncell aggressiveness, we decided to perform in vitro experiments upon activation of this\\nreceptor. To select an appropriate cellular model, we analyzed the levels of ERBB3 mRNA\\nin 51 breast cell lines, which were previously characterized and divided into luminal and\\nbasal-like (basal A and basal B) subgroups [ 29]. Basal-like cell lines, in particular within\\nthe basal B subgroup, showed lower ERBB3 mRNA levels compared to luminal cell lines\\n(Supplementary Figure S2a,b), mirroring the observation in basal-like breast cancer patients.\\nIn accordance, triple-negative breast cancer exhibits low expression levels of ERBB3 com-\\npared to the other clinical subtypes (Supplementary Figure S2c). Furthermore, a modest\\nincrease in NRG1 mRNA levels was observed in the basal-B subgroup (Supplementary\\nFigure S2d,e), while the stratiﬁcation for clinical breast cancer subtypes highlighted a high\\nvariability in the triple-negative subgroup without signiﬁcant differences with the other sub-\\ntypes (Supplementary Figure S2f). Among basal-like cell lines, we then selected MCF10A\\ncells as the cellular model. These cells were derived from benign breast tissue sponta-\\nneously immortalized without deﬁned factors [ 34]. Thus, the employment of MCF10A\\ncells as a cellular model avoids the interference of multiple aberrations usually occurring in\\ncancer cells. In addition, MCF10A cells are a widely employed and accepted model to study\\nthe mechanistic contribution of different ERBB family members [ 31,32,35,36]. MCF10A\\ncells, as most basal B cell lines, express very low levels of ERBB2 (Supplementary Figure\\nS3a–c), ER-alpha (Supplementary Figure S3d–f), and PR (Supplementary Figure S3g–i),\\nthus they are also considered “triple-negative” [37] (reviewed in [38]).\\nTo evaluate a potential role for ERBB3 signaling in basal-like/triple-negative breast\\ncells, we employed the ERBB3 ligand neuregulin 1 β(NRG1 β). Firstly, MCF10A cells were\\ntreated with medium containing NRG1 β(10 ng/mL) or with control medium for seven\\ndays, then ﬁxed and analyzed for cell cycle activity by immunostaining of KI67 protein.\\nIndeed, nuclear KI67 immunoreactivity is detectable during all the active phases of the cell\\ncycle (G1, S, G2, and M phases), but not in resting cells (G0 phase). However, our results\\ndid not unveil signiﬁcant differences in the percentage of cells in the active phases of the cell\\ncycle following stimulation with NRG1 β(Figure 3a). To better evaluate cell proliferation\\nupon administration of NRG1 β, we then employed the Livecyte®Cell Analysis System,\\nwhich provides high contrast time-lapse videos with automated live cell analysis. To this\\nend, MCF10A cells were treated with medium containing NRG1 β(10 ng/mL) or with\\ncontrol medium, and then their number and morphological features were followed over\\ntime up to 72 h. This analysis did not show signiﬁcant differences in cell number upon\\nadministration of NRG1 β(Figure 3b and Supplementary Figure S4). We thus concluded\\nthat NRG1 βdoes not play a role in the proliferation of MCF10A cells. After stimulation\\nwith NRG1 β, we also did not observe signiﬁcant differences in the instantaneous velocity\\nof cells up to 12 h (Figure 3c). Moreover, no major differences in random versus directional\\nmigration were observed, as evidenced by similar cell displacement (Figure 3d) and con-\\nﬁnement ratio (Figure 3e,f) at different time points (6 and 12 h). Overall, our data suggest\\nthat the NRG1 β/ERBB3/ERBB2 axis does not play a role in cell proliferation or motility of\\nbasal-like breast cells cultured in monolayer conditions.Cancers 2022 ,14, 1603 9 of 19\\nCancers 2022 , 14, x 9 of 20 \\n \\n of the cell cycle following stimulation with NRG1 β (Figure 3a). To better evaluate cell \\nproliferation upon administration of NRG1 β, we then employed the Livecyte® Cell Anal-\\nysis System, which provides high contrast time -lapse videos with automated live cell anal-\\nysis. To this end, MCF10A cells were  treated with medium containing NRG1 β (10 ng/mL) \\nor with control medium, and then their nu mber and morphological features were fol-\\nlowed over time up to 72 h. This analysis did not show significant differences in cell num-\\nber upon administration of NRG1 β (Figure 3b and Supplementary Figure S4). We thus \\nconcluded that NRG1 β does not play a role in the proliferation of MCF10A cells. After \\nstimulation with NRG1 β, we also did not observe significant differences in the instanta-\\nneous velocity of cells up to 12 h (Figure 3c). Moreover, no major differences in random \\nversus directional migration were observed, as evidenced by similar cell displacement \\n(Figure 3d) and confinement ratio (Figure 3e,f) at different time points (6 and 12 h). Over-all, our data suggest that the NRG1 β/ERBB3/ERBB2 axis does not play a role in cell pro-\\nliferation or motility of basal-like breast cells cultured in monolayer conditions. \\n \\nFigure 3. Proliferation and migration of basal-like/triple -negative breast cells cultured in monolayer \\nconditions are not affected by the administration of the growth factor NRG1 β. (a) Immunofluores-\\ncence analysis of KI67 for the evalua tion of cells in the active phases of the cell cycle. MCF10A cells \\nwere cultured in vitro and treated with/without NRG1 β (10 ng/mL) for seven days. Subsequently, \\nthey were fixed and stained with an anti-KI67 an tibody and DAPI (for nucl ear staining) and exam-\\nined with an inverted fluorescent microscope. Photoshop software was used to quantify KI67 posi-\\ntive cells of three different images taken for each sample. Representative pictures are provided; scale \\nbar, 10 µm. The histograms show the analysis of positively stained cells (means ± SEM) for MCF10A \\ncontrol and NRG1-treated cells, respectively. ( b) Cell count analysis over time by Livecyte technol-\\nogy. MCF10A were cultured in vitro and treated with/without NRG1 β (10 ng/mL) and monitored \\nu p  t o  7 2  h .  ( c) Cell speed analysis over time by Live cyte technology of MCF10A cells treated \\nwith/without NRG1 β (10 ng/mL). Cell velocity has been detected every hour up to 12 h. ( d) MCF10A \\ncell displacement analysis by Livecyte technology.  Representative track plots of MCF10A treated \\nwith/without NRG1 β (10 ng/mL) up to 6 and 12 h are provided. ( e,f) Confinement ratio analysis of \\nFigure 3. Proliferation and migration of basal-like/triple-negative breast cells cultured in monolayer\\nconditions are not affected by the administration of the growth factor NRG1 β. (a) Immunoﬂuores-\\ncence analysis of KI67 for the evaluation of cells in the active phases of the cell cycle. MCF10A cells\\nwere cultured in vitro and treated with/without NRG1 β(10 ng/mL) for seven days. Subsequently,\\nthey were ﬁxed and stained with an anti-KI67 antibody and DAPI (for nuclear staining) and examined\\nwith an inverted ﬂuorescent microscope. Photoshop software was used to quantify KI67 positive cells\\nof three different images taken for each sample. Representative pictures are provided; scale bar, 10 µm.\\nThe histograms show the analysis of positively stained cells (means ±SEM) for MCF10A control and\\nNRG1-treated cells, respectively. ( b) Cell count analysis over time by Livecyte technology. MCF10A\\nwere cultured in vitro and treated with/without NRG1 β(10 ng/mL) and monitored up to 72 h.\\n(c) Cell speed analysis over time by Livecyte technology of MCF10A cells treated with/without\\nNRG1 β(10 ng/mL). Cell velocity has been detected every hour up to 12 h. ( d) MCF10A cell displace-\\nment analysis by Livecyte technology. Representative track plots of MCF10A treated with/without\\nNRG1 β(10 ng/mL) up to 6 and 12 h are provided. ( e,f) Confinement ratio analysis of MCF10A cells by\\nLivecyte technology . Violin plots of MCF10A treated with/without NRG1 β(10 ng/mL) for 6 and 12 h.\\n3.3. Administration of NRG1 βDoes Not Signiﬁcantly Impact Cell Differentiation in\\nBasal-like/Triple-Negative Breast Cells\\nUndifferentiated or poorly differentiated cancer cells tend to grow and spread at a\\nfaster rate than well-differentiated cancer cells. We thus analyzed if ERBB3 activation\\nby NRG1 βmay reduce basal-like breast cell differentiation. Our initial Livecyte analysis\\nof morphological parameters of NRG1 β-treated versus control MCF10A cells showed\\nminimal and not statistically signiﬁcant variations in terms of cell size (Figure 4a), sphericity\\n(Figure 4b), and length-to-width ratio (Figure 4c).\\nThe mammary epithelium consists of differentiated cell types organized into two cell\\nlayers, an inner layer of luminal (ductal and alveolar) epithelial cells surrounded by an\\nouter layer of myoepithelial cells in direct contact with the basement membrane [ 39,40]\\n(reviewed in [ 41–44]). Myoepithelial and luminal cells typically express different sets ofCancers 2022 ,14, 1603 10 of 19\\nkeratins. In particular, myoepithelial cells express cytokeratin 5/6 ( KRT5 ), cytokeratin\\n14 (KRT14 ), and cytokeratin 17 ( KRT17 ), whereas luminal cells express cytokeratin 8 ( KRT8 ),\\ncytokeratin 18 ( KRT18 ), and cytokeratin 7 ( KRT7 ) [39,40] (reviewed in [ 42–45]). Basal-\\nlike breast cancer is deﬁned by the expression of genes abundantly expressed by basal\\nmyoepithelial cells [ 16,17,46]. Accordingly, mRNA expression of myoepithelial cytokeratins\\n(KRT5, KRT14 , and KRT17 ) was upregulated in our datasets of primary basal-like tumors\\n(Supplementary Figure S5a–c) and cell lines (Supplementary Figure S6a–c). Similarly, triple-\\nnegative cell lines also expressed higher mRNA levels of these keratins (Supplementary\\nFigure S6d–f). Furthermore, high levels of expression of these basal/myoepithelial keratins\\nwere observed in MCF10A cells (Supplementary Figure S6g–i), again conﬁrming these cells\\nas a suitable basal-like/triple-negative model.\\nCancers 2022 , 14, x 11 of 20 \\n \\n  \\nFigure 4. Administration of NRG1 β does not influence morphological parameters and the expres-\\nsion of basal/myoepithelial and luminal differentia tion markers in basal-lik e/triple-negative breast \\ncells. ( a–c) Analysis of morphological parameters, namely ( a) cell size, ( b) sphericity, and ( c) length-\\nto-width ratio in MCF10A cells cultured  in vitro and treated with/without NRG1 β (10 ng/mL) for \\n72 h; ( d,e) mRNA expression levels of myoepithelial/basal markers cytokeratin 5/6 ( d, KRT5 ) and \\ncytokeratin 14 ( e, KRT14 ) in MCF10A cells cultured in vitro and treated with/without NRG1 β (10 \\nng/mL); ( f) mRNA expression levels of TP63  gene in MCF10A cells cultured in vitro and treated \\nwith/without NRG1 β (10 ng/mL); ( g,h) mRNA expression levels of luminal markers cytokeratin 8 \\n(g, KRT8 ), and mucin1 ( h, MUC1 ) in MCF10A cells cultured in vitro and treated with/without \\nNRG1β (10 ng/mL). In all panels, numerical data are pr esented as the mean (erro r bars show s.e.m.). \\n3.4. NRG1 β/ERBB3/ERBB2 Robustly Promotes Anchorage-Independent Cell Growth of Basal-\\nlike Breast Cells \\nOur clinical data showed a correlation between higher ERBB3  mRNA expression lev-\\nels and shorter relapse-free survival in basal-like breast cancer patients. Since our initial \\nanalyses did not unveil significant effects induced by NRG1 β on cell proliferation, differ-\\nentiation, and migration of basa l-like/triple-negative cells, we decided to test the role of \\nthe NRG1 β/ERBB3/ERBB2 axis in non-adherent cond itions. To this aim, cells were seeded \\nin ultra-low attachment plates, which feature a covalently bound hydrogel layer that ef-fectively inhibits cellular attachment. At first, we measured cell survival in response to \\nNRG1β stimulation by flow cytometry analysis of Annexin V/propidium iodide (PI) stain-\\ning. In particular, Annexin V binds to cells in early apoptosis, whereas the PI binds to \\nDNA into the nucleus in the late stages of cellular apoptosis and to necrotic cells. How-\\never, no appreciable effects upon stimulation with NRG1 β were observed at all analyzed \\ntime points, namely 24, 48, and 72 h after the stimulation (Supplementary Figure S8a–c). \\nAfterward, we evaluated the impact of the NRG1 β/ERBB3/ERBB2 axis in promoting cell \\ngrowth in non-adherent conditions. To this en d, cells were seeded in ultra-low attachment \\nplates and every three days were treated with NRG1 β, by replacing half of the medium. \\nFigure 4. Administration of NRG1 βdoes not inﬂuence morphological parameters and the expression\\nof basal/myoepithelial and luminal differentiation markers in basal-like/triple-negative breast cells.\\n(a–c) Analysis of morphological parameters, namely ( a) cell size, ( b) sphericity, and ( c) length-to-\\nwidth ratio in MCF10A cells cultured in vitro and treated with/without NRG1 β(10 ng/mL) for 72 h;\\n(d,e) mRNA expression levels of myoepithelial/basal markers cytokeratin 5/6 ( d)KRT5 and cytoker-\\natin 14 ( e)KRT14 in MCF10A cells cultured in vitro and treated with/without NRG1 β(10 ng/mL );\\n(f) mRNA expression levels of TP63 gene in MCF10A cells cultured in vitro and treated with/without\\nNRG1 β(10 ng/mL ); (g,h) mRNA expression levels of luminal markers cytokeratin 8 ( g)KRT8 , and\\nmucin1 ( h)MUC1 in MCF10A cells cultured in vitro and treated with/without NRG1 β(10 ng/mL).\\nIn all panels, numerical data are presented as the mean (error bars show s.e.m.).\\nTo evaluate the potential impact of the NRG1 β/ERBB3/ERBB2 axis on basal/\\nmyoepithelial cell differentiation, we analyzed the expression of basal markers in MCF10A\\ncells exposed to NRG1 β. Interestingly, administration of NRG1 βto MCF10A did not\\ninduce signiﬁcant changes in KRT5 and KRT14 mRNA expression (Figure 4d,e). Another\\nproposed marker for myoepithelial/basal cells and basal-like tumors is p63 [ 40] (reviewed\\nin [42]). Surprisingly, in our meta-analyses, the expression of p63 gene ( TP63 ) was sig-Cancers 2022 ,14, 1603 11 of 19\\nniﬁcantly increased neither in basal-like tumors (Supplementary Figure S7a) nor in basal\\nor triple-negative lines (Supplementary Figure S7b,c). Nevertheless, TP63 mRNA was\\nhighly expressed in MCF10A cells (Supplementary Figure S7d) and was not signiﬁcantly\\nregulated by exposure to NRG1 β(Figure 4f). On the other hand, NRG1 βtreatment did\\nnot inﬂuence the mRNA expression of luminal markers, such as cytokeratin 8 ( KRT8 ) and\\nmucin 1 ( MUC1 ) (Figure 4g,h).\\nOverall, our data suggest that cell differentiation of basal-like/triple-negative cells is\\nnot signiﬁcantly regulated by the NRG1 β/ERBB3/ERBB2 axis.\\n3.4. NRG1 β/ERBB3/ERBB2 Robustly Promotes Anchorage-Independent Cell Growth of Basal-like\\nBreast Cells\\nOur clinical data showed a correlation between higher ERBB3 mRNA expression\\nlevels and shorter relapse-free survival in basal-like breast cancer patients. Since our initial\\nanalyses did not unveil signiﬁcant effects induced by NRG1 βon cell proliferation, dif-\\nferentiation, and migration of basal-like/triple-negative cells, we decided to test the role\\nof the NRG1 β/ERBB3/ERBB2 axis in non-adherent conditions. To this aim, cells were\\nseeded in ultra-low attachment plates, which feature a covalently bound hydrogel layer\\nthat effectively inhibits cellular attachment. At ﬁrst, we measured cell survival in response\\nto NRG1 βstimulation by ﬂow cytometry analysis of Annexin V/propidium iodide (PI)\\nstaining. In particular, Annexin V binds to cells in early apoptosis, whereas the PI binds to\\nDNA into the nucleus in the late stages of cellular apoptosis and to necrotic cells. How-\\never, no appreciable effects upon stimulation with NRG1 βwere observed at all analyzed\\ntime points, namely 24, 48, and 72 h after the stimulation (Supplementary Figure S8a–c).\\nAfterward, we evaluated the impact of the NRG1 β/ERBB3/ERBB2 axis in promoting cell\\ngrowth in non-adherent conditions. To this end, cells were seeded in ultra-low attachment\\nplates and every three days were treated with NRG1 β, by replacing half of the medium.\\nThe number and size of spheroids, namely three-dimensional (3D) cell cultures that arrange\\nthemselves during proliferation into sphere-like formations, were then measured after\\n14 days in culture. Strikingly, in contrast to 2D monolayer cell condition, NRG1 βpromoted\\nan increase in both spheroid number and size (Figure 5a,b). Interestingly, MCF10A cells\\ncultured in 3D compared to 2D conditions strongly upregulated ERBB3 and ERBB2 pro-\\ntein levels (Figure 5c). These data suggest a role for ERBB3 and ERBB2 in mediating the\\nanchorage-independent growth induced by NRG1 βin basal-like/triple-negative breast\\ncells. Next, we sought to neutralize ERBB2, by employing two anti-ERBB2 agents, namely\\ntrastuzumab and pertuzumab, currently used in clinics. Trastuzumab is used to treat HER2-\\npositive breast cancer patients in metastatic and adjuvant settings (reviewed in [ 1,47]). It\\nis a humanized monoclonal antibody that binds to the extracellular domain IV of HER2,\\nstrongly inhibiting its ligand-independent activation, which has been reported to mainly\\noccur when ERBB2 is overexpressed [ 48,49] (reviewed in [ 1,47,50]). Pertuzumab has shown\\nantitumor activity in both the metastatic and the neoadjuvant settings of HER2-positive\\nbreast cancer patients and has been more recently approved as adjuvant therapy. It is a\\nhumanized monoclonal antibody that binds to the extracellular domain II of HER2, essen-\\ntial for dimerization (reviewed in [ 47,50]). Thus, pertuzumab was suggested to efﬁciently\\ninhibit ligand-dependent HER2 dimerization. In contrast, trastuzumab was reported to be\\nless effective in the presence of ERBB ligands [ 51] (reviewed in [ 47]). Co-administration\\nof trastuzumab or pertuzumab was therefore used to evaluate the contribution of ERBB2\\nto NRG1 βsignaling in anchorage-independent growth (Figure 5d). In our experiments,\\nadministration of trastuzumab was not efﬁcient in reducing the increase in spheroids’\\nformation driven by NRG1 β, even if a trend toward a reduction in spheroids’ formation\\nwas observed when administered alone (Figure 5e). Furthermore, trastuzumab, alone or in\\ncombination with NRG1 β, was unable to reduce spheroid’s size (Figure 5f). In contrast,\\npertuzumab completely suppressed the action of NRG1 β, despite not showing any effect\\non spheroids’ number or size when administered alone (Figure 5d–f).Cancers 2022 ,14, 1603 12 of 19\\nCancers 2022 , 14, x 13 of 20 \\n \\n  \\nFigure 5. Administration of NRG1 β promotes the anchorage-indepe ndent growth of basal-like/tri-\\nple-negative breast cells. ( a,b) Spheroid forming efficiency and gr owth (size) of MCF10A cells upon \\nstimulation with NRG1 β. MCF10A cells were seeded in ultra- low attachment plates and stimulated \\nwith/without NRG1 β (10 ng/mL) every 3 days for up to 14 days. Spheroid number (spheroid form-\\ning efficiency) and size are provided in ( a,b), respectively. ( c) Western blot analysis of ERBB3 and \\nERBB2 protein levels in lysates of MCF10A cells cultured in monolayer (2D) and non-adherent con-\\nditions (3D). ( d–f) 3D cell growth of MCF10A cells upon stimulation with NRG1 β and simultaneous \\ninhibition of ERBB2 by trastuzumab (TZ) or pertuzumab (PZ). MCF10A cells were seeded in ultra-\\nlow attachment plates and stimulated with/without NRG1 β (10 ng/mL), alone or in combination \\nwith trastuzumab (10 µg/mL) or pertuzumab (10 µg /mL). Stimulations were repeated every 3 days \\nup to 14 days. Representative pictures are provided in ( d). Spheroid number (spheroid forming ef-\\nficiency) and size (spheroid growth) are provided in ( e,f), respectively. In all panels, numerical data \\nare normalized to control cells and presented as the mean (error bars show s.e. m.); statistical signif-\\nicance was determined using Student’s t-test in ( a,b), and one-way ANOVA followed by Tukey’s \\ntest in ( e,f); * p < 0.05, ** p < 0.01, *** p < 0.001, and **** p < 0.0001. \\nFigure 5. Administration of NRG1 βpromotes the anchorage-independent growth of basal-like/triple-\\nnegative breast cells. ( a,b) Spheroid forming efﬁciency and growth (size) of MCF10A cells upon\\nstimulation with NRG1 β. MCF10A cells were seeded in ultra-low attachment plates and stimulated\\nwith/without NRG1 β(10 ng/mL) every 3 days for up to 14 days. Spheroid number (spheroid\\nforming efﬁciency) and size are provided in ( a,b), respectively. ( c) Western blot analysis of ERBB3 and\\nERBB2 protein levels in lysates of MCF10A cells cultured in monolayer (2D) and non-adherent condi-\\ntions (3D). ( d–f) 3D cell growth of MCF10A cells upon stimulation with NRG1 βand simultaneous\\ninhibition of ERBB2 by trastuzumab (TZ) or pertuzumab (PZ). MCF10A cells were seeded in ultra-low\\nattachment plates and stimulated with/without NRG1 β(10 ng/mL), alone or in combination with\\ntrastuzumab (10 µg/mL) or pertuzumab (10 µg/mL). Stimulations were repeated every 3 days up to\\n14 days. Representative pictures are provided in ( d). Spheroid number (spheroid forming efﬁciency)\\nand size (spheroid growth) are provided in ( e,f), respectively. In all panels, numerical data are\\nnormalized to control cells and presented as the mean (error bars show s.e.m.); statistical signiﬁcance\\nwas determined using Student’s t-test in ( a,b), and one-way ANOVA followed by Tukey’s test in ( e,f);\\n*p< 0.05, ** p< 0.01, *** p< 0.001, and **** p< 0.0001.\\nSince MCF10A cells are considered a quasi-normal basal-like/triple-negative cellular\\nmodel, we decided to validate our ﬁndings on basal-like/triple-negative breast cancer cells.\\nTo this end, we employed MDA-MB-468 and MDA-MB-231 cells. In line with the data\\nobtained on MCF10A cells, administration of NRG1 βdid not signiﬁcantly induce cell prolif-\\neration when MDA-MB-231 and MDA-MB-468 cells were cultured in adhesion ( Figure 6a,b );\\nhowever, it promoted their cell growth when cultured in anchorage-independent condi-\\ntions (Figure 6c,d). Interestingly, in MDA-MB-468, this process was inhibited by bothCancers 2022 ,14, 1603 13 of 19\\ntrastuzumab and pertuzumab, whereas in MDA-MB-231, we observed a trend of reduction\\nonly with pertuzumab (Figure 6c,d).\\nCancers 2022 , 14, x 14 of 20 \\n \\n  \\nFigure 6. Administration of NRG1 β promotes the anchorage-indepe ndent growth of basal-like/tri-\\nple-negative breast cancer cells. ( a,b) Anchorage-dependent cell proliferation of MDA-MB-468 and \\nMDA-MB-231 cells upon stimulation with NRG1 β. MDA-MB-468 and MDA-MB-231 cells were \\nseeded in adherent co nditions and stimulated with/without NRG1 β (10 ng/mL) for 72 h. Cell count \\nfor MDA-MB-468 and MDA-MB-231 is provided in ( a) and ( b), respectively. ( c,d) Anchorage-inde-\\npendent cell growth of MDA-MB-468 and MD A-MB-231 cells upon stimulation with NRG1 β and \\nsimultaneous inhibition of ERBB2 by trastuzu mab (TZ) or pertuzumab (PZ). MDA-MB-468 and \\nMDA-MB-231 cells were seeded in ultra-low atta chment plates and stimulated with/without \\nNRG1β (10 ng/mL), alone or in combination with trastuzumab (10 µg/mL) or pertuzumab (10 \\nµg/mL). Stimulations were repeated every 3 days fo r up to 14 days. Representative pictures and cell \\ncount for MDA-MB-468 and MDA-MB-231 are provided in ( c) and ( d), respectively. In all panels, \\nnumerical data are normalized to control cells and presented as mean (error bars show s.e.m.); sta-\\ntistical significance was determined using Student’s t-test in ( a,b), and one-way ANOVA followed \\nby Tukey’s test in ( c,d); * p < 0.05, ** p < 0.01, *** p < 0.001, and **** p < 0.0001. \\n3.5. Pertuzumab Efficiently Blocks NRG1 β-Induced Cell Growth in Non- Adherent Conditions in \\nHER2-Overexpressing Basal-like Breast Cells \\nAbout 25% of triple-negative breast cancers with HER2-negative expression in the \\nprimary tumor convert to HER2-low expression  in the recurrent tumor, maintaining the \\nsame cancer phenotype [52]. Our study suggests that the loss of cell adhesion may con-\\ntribute to ERBB2 upregulation. To evaluate the role of the NRG1 β/ERBB3/ERBB2 axis in \\nbasal-like cells with increased expression of  ERBB2, we also analyzed previously gener-\\nated MCF10A cells stably overexpressing ERBB2 (MCF10A-HER2) [31,32]. Administra-\\ntion of NRG1 β to MCF10A-HER2 induced an increase in spheroid’s number and size com-\\npared to untreated control cells, which was significantly prevented by pertuzumab, but \\nnot by trastuzumab co-administration (Figure 7a–c). Overall, our data suggest that per-\\ntuzumab efficiently inhibits the growth of basal-like breast cells induced by NRG1 β, even \\nwhen ERBB2 is upregulated. \\nFigure 6. Administration of NRG1 βpromotes the anchorage-independent growth of basal-like/triple-\\nnegative breast cancer cells. ( a,b) Anchorage-dependent cell proliferation of MDA-MB-468 and MDA-\\nMB-231 cells upon stimulation with NRG1 β. MDA-MB-468 and MDA-MB-231 cells were seeded in\\nadherent conditions and stimulated with/without NRG1 β(10 ng/mL) for 72 h. Cell count for MDA-\\nMB-468 and MDA-MB-231 is provided in ( a) and ( b), respectively. ( c,d) Anchorage-independent cell\\ngrowth of MDA-MB-468 and MDA-MB-231 cells upon stimulation with NRG1 βand simultaneous\\ninhibition of ERBB2 by trastuzumab (TZ) or pertuzumab (PZ). MDA-MB-468 and MDA-MB-231 cells\\nwere seeded in ultra-low attachment plates and stimulated with/without NRG1 β(10 ng/mL), alone\\nor in combination with trastuzumab (10 µg/mL) or pertuzumab (10 µg/mL). Stimulations were\\nrepeated every 3 days for up to 14 days. Representative pictures and cell count for MDA-MB-468 and\\nMDA-MB-231 are provided in ( c) and ( d), respectively. In all panels, numerical data are normalized\\nto control cells and presented as mean (error bars show s.e.m.); statistical signiﬁcance was determined\\nusing Student’s t-test in ( a,b), and one-way ANOVA followed by Tukey’s test in ( c,d); *p< 0.05,\\n**p< 0.01, *** p< 0.001, and **** p< 0.0001.\\nOverall, our data unveil that the NRG1 β/ERBB3/ERRB2 axis promotes cell growth in\\nsuspension of basal-like/triple-negative breast cancer cells and that this process is efﬁciently\\nprevented by the administration of anti-ERBB2 agents, in particular, pertuzumab.\\n3.5. Pertuzumab Efﬁciently Blocks NRG1 β-Induced Cell Growth in Non-Adherent Conditions in\\nHER2-Overexpressing Basal-like Breast Cells\\nAbout 25% of triple-negative breast cancers with HER2-negative expression in the\\nprimary tumor convert to HER2-low expression in the recurrent tumor, maintaining the\\nsame cancer phenotype [ 52]. Our study suggests that the loss of cell adhesion may con-\\ntribute to ERBB2 upregulation. To evaluate the role of the NRG1 β/ERBB3/ERBB2 axis in\\nbasal-like cells with increased expression of ERBB2, we also analyzed previously generatedCancers 2022 ,14, 1603 14 of 19\\nMCF10A cells stably overexpressing ERBB2 (MCF10A-HER2) [ 31,32]. Administration of\\nNRG1 βto MCF10A-HER2 induced an increase in spheroid’s number and size compared\\nto untreated control cells, which was signiﬁcantly prevented by pertuzumab, but not by\\ntrastuzumab co-administration (Figure 7a–c). Overall, our data suggest that pertuzumab\\nefﬁciently inhibits the growth of basal-like breast cells induced by NRG1 β, even when\\nERBB2 is upregulated.\\nCancers 2022 , 14, x 15 of 20 \\n \\n  \\n \\nFigure 7. Administration of NRG1 β promotes growth in non-adherent conditions in basal-like/tri-\\nple-negative breast cells engineer ed with HER2 overexpression. ( a–c) Spheroid forming efficiency \\nand growth (size) of HER2-overexpressing MCF 10A (MCF10-HER2) upon stimulation with NRG1 β. \\nMCF10A cells were seeded in ultra-low attach ment plates and stimulated with/without NRG1 β (10 \\nng/mL), trastuzumab (TZ, 10 µg/mL)  or pertuzumab (PZ, 10 µg/mL) every 3 days up to 14 days. \\nRepresentative pictures of spheroids are provided in ( a); spheroids number (spheroid forming effi-\\nciency) and size are provided in ( b,c), respectively. In all panels, numerical data are normalized to \\ncontrol cells and presented as mean  (error bars show s.e.m.); statistical significance was determined \\nusing one-way ANOVA followed by Tukey’s test in ( b,c); * p < 0.05, ** p < 0.01, *** p < 0.001, and **** \\np < 0.0001. \\n4. Discussion \\nIn this stud y,  we report an inverse  cor relation between ERBB3  mRNA expression \\nlevels and relapse-free survival in basal-like breast cancer patients. Of note, although \\nERBB3 role in breast cancer progression is we ll established, our work suggests its specific \\ninvolvement in basal-like/triple- negative breast cancer cells. By employing the growth fac-\\ntor neuregulin 1 β (NRG1β), we analyzed the impact of ERBB3 signaling activation on \\nmultiple phenotypes in a basal-like quasi-normal breast cellular model (MCF10A cells). \\nNotably, our data demonstrate that NRG1 β administration strongly increases basal-\\nlike/triple-negative cell growth as spheroids in non-adherent conditions. Importantly, the \\nability to grow in anchorage-independent cond itions, thus forming spheroids, is thought \\nto be responsible for metastatic tumor dissemination, which in turn is the cause of death \\nin more than 90% of human cancer patients [53]. Metastasis is indeed a multistep process \\nin which cancer cells invade the basement membrane, dissociate from primary sites, in-\\ntravasate in the vascular or lymphatic system, survive to the circulation, extravasate from \\nthe vasculature to secondary tissue, and finally  proliferate in distant organs. As a barrier \\nto metastasis, cells normally undergo an apoptotic process known as “anoikis,” a form of \\ncell death consequent to loss of adhesion with  the extracellular matrix or with neighboring \\ncells. Cancer cells acquire anoikis resistance to survive after detachment from the primary \\nsites and travel through the circulatory and lymphatic systems to disseminate throughout \\nthe body. Intriguingly, NRG1 β administration was unable to support cell survival of ba-\\nFigure 7. Administration of NRG1 βpromotes growth in non-adherent conditions in basal-like/triple-\\nnegative breast cells engineered with HER2 overexpression. ( a–c) Spheroid forming efﬁciency and\\ngrowth (size) of HER2-overexpressing MCF10A (MCF10-HER2) upon stimulation with NRG1 β.\\nMCF10A cells were seeded in ultra-low attachment plates and stimulated with/without NRG1 β\\n(10 ng/mL ), trastuzumab (TZ, 10µg/mL ) or pertuzumab (PZ, 10µg/mL ) every 3 days up to 14 days .\\nRepresentative pictures of spheroids are provided in ( a); spheroids number (spheroid forming\\nefﬁciency) and size are provided in ( b,c), respectively. In all panels, numerical data are normalized to\\ncontrol cells and presented as mean (error bars show s.e.m.); statistical signiﬁcance was determined\\nusing one-way ANOVA followed by Tukey’s test in ( b,c); *p< 0.05, ** p< 0.01, *** p< 0.001, and\\n****p< 0.0001.\\n4. Discussion\\nIn this study, we report an inverse correlation between ERBB3 mRNA expression levels\\nand relapse-free survival in basal-like breast cancer patients. Of note, although ERBB3 role\\nin breast cancer progression is well established, our work suggests its speciﬁc involvement\\nin basal-like/triple-negative breast cancer cells. By employing the growth factor neuregulin\\n1β(NRG1 β), we analyzed the impact of ERBB3 signaling activation on multiple phenotypes\\nin a basal-like quasi-normal breast cellular model (MCF10A cells). Notably, our data\\ndemonstrate that NRG1 βadministration strongly increases basal-like/triple-negative cell\\ngrowth as spheroids in non-adherent conditions. Importantly, the ability to grow in\\nanchorage-independent conditions, thus forming spheroids, is thought to be responsible\\nfor metastatic tumor dissemination, which in turn is the cause of death in more than 90%\\nof human cancer patients [ 53]. Metastasis is indeed a multistep process in which cancer\\ncells invade the basement membrane, dissociate from primary sites, intravasate in the\\nvascular or lymphatic system, survive to the circulation, extravasate from the vasculature\\nto secondary tissue, and ﬁnally proliferate in distant organs. As a barrier to metastasis,\\ncells normally undergo an apoptotic process known as “anoikis,” a form of cell deathCancers 2022 ,14, 1603 15 of 19\\nconsequent to loss of adhesion with the extracellular matrix or with neighboring cells.\\nCancer cells acquire anoikis resistance to survive after detachment from the primary sites\\nand travel through the circulatory and lymphatic systems to disseminate throughout the\\nbody. Intriguingly, NRG1 βadministration was unable to support cell survival of basal-like\\nbreast cells cultured in suspension. Thus, our data suggest that NRG1 βmay speciﬁcally\\nsupport the development and growth of metastasis in basal-like breast cancer patients.\\nIn our study, the activation of the NRG1 β/ERBB3/ERBB2 axis in basal-like/triple-\\nnegative breast cells cultured in monolayer did not induce cell proliferation and motility,\\nand did not signiﬁcantly impact cell differentiation. Nevertheless, we cannot exclude the\\npossibility that chronic exposure to NRG1 βwould impact these phenotypes in the long\\nterm. In addition, our study did not evaluate the potential role of NRG1 directly produced\\nby cancer cells. In this regard, silencing of NRG1 in triple-negative cellular models has\\nbeen very recently documented to reduce cell migration [ 54], further supporting a role for\\nNRG1 signaling in basal-like/triple-negative cancers. Previously, NRG1 administration\\nto MCF10A cells engineered with ERBB2 overexpression was demonstrated to induce an\\ninvasive phenotype with protruded arms when cultured in 3D conditions [ 32]. Stimulation\\nwith neuregulin 1 after combinatorial overexpression of wild-type ERBB3 and ERBB2 has\\nbeen shown to promote anchorage-independent growth in MCF10A cells [ 55]. Moreover,\\noverexpression of active ERBB3 mutants and wild-type ERBB2 promoted similar effects in\\na ligand-independent manner [ 6]. Our data in MCF10A cells suggest that NRG1 βis more\\neffective in supporting 3D compared to 2D cell growth, due to an upregulation in ERBB3\\nand ERBB2 receptor abundance when cells are grown in anchorage-independent conditions.\\nThis is particularly interesting considering that it has been reported that about 36% of triple-\\nnegative breast cancers can switch from HER2-negative to HER2-low expression, whereas\\nabout 5% can switch from HER2-negative to HER2-positive [ 52]. Our work suggests that\\nthe loss of cell adhesion, which occurs after intravasation in the vascular or lymphatic\\nsystem during the metastatic process, may contribute to these transitions. Nevertheless,\\npreliminary analyses did not conﬁrm this upregulation in MDA-MB-468 and MDA-MB-231\\nwhen cultured in anchorage-independent conditions (data not shown), suggesting that this\\nmechanism may be inﬂuenced by speciﬁc genetic alterations occurring in cancer cells.\\nThe role of ERBB2 in cancer progression is well established (reviewed in [ 1,5]) and\\ntargeting ERBB2 through speciﬁc monoclonal antibodies, such as trastuzumab and per-\\ntuzumab, represents the gold standard to treat HER2-positive breast cancer patients in\\nclinics (reviewed in [ 1]). So far, only cancer patients with strong HER2 expression are\\ntreated with anti-HER2 drugs. Indeed, in common clinical practice, the treatment-decision\\nprocess is driven by the dichotomization in HER2-positive vs. negative, with HER2-positive\\ndeﬁned by a score of 3+ HER2 protein overexpression in IHC analysis and/or HER2 gene\\nampliﬁcation in an ISH assay. However, HER2-negative breast cancers are characterized\\nby a wide spectrum of HER2 expression levels, with about half of breast cancers classiﬁed\\nas HER2-negative showing low HER2 expression [ 52]. Notably, preliminary results from\\nearly phase clinical trials testing trastuzumab-drug conjugates in advanced breast cancer\\npatients harboring HER2-low expression were promising [ 56,57] (reviewed in [ 58]). On\\nthe other hand, a phase II trial showed disappointing tumor responses upon pertuzumab\\nadministration alone to HER2-negative or HER2-low breast cancer patients [ 59] (reviewed\\nin [58]). Based on our data, we here suggest targeting the NRG1/ERBB3/ERBB2 signaling\\naxis in basal-like/triple-negative breast tumors. The oncogenic role of ERBB2 has been\\npoorly investigated in basal-like/triple-negative breast cancers owing to the low expression\\nof this receptor in these breast cancer subtypes. Despite that, quite recently the canonical\\nERBB2 isoform and an ERBB2 variant located in the nucleus were suggested to drive triple-\\nnegative breast cancer growth [ 60]. In our study, administration of pertuzumab was able to\\nsuppress spheroid’s forming efﬁciency and growth driven by NRG1 βstimulation in all an-\\nalyzed basal-like/triple-negative cellular models, whereas trastuzumab was effective only\\nin MDA-MB-468 cells. Even if the differences in trastuzumab response in other basal-like\\ncell lines deserve further investigations, our data are mostly in line with recent literatureCancers 2022 ,14, 1603 16 of 19\\nsuggesting that pertuzumab more efﬁciently blocks ligand-induced ERBB2 activation [ 51]\\n(reviewed in [ 47]). We thus suggest the speciﬁc administration of pertuzumab to basal-like\\nbreast cancer patients, which, to our knowledge, has not been yet tested. Because dual\\nHER2-targeted therapy has been reported to improve overall survival and progression-free\\nsurvival compared to single HER2-targeted therapy [ 61] (reviewed in [ 62]), we also suggest\\nevaluation of this combinatorial therapy as a treatment for basal-like/triple-negative breast\\ncancer patients.\\n5. Conclusions\\nOur study unveils a link between high ERBB3 expression and poor relapse-free pa-\\ntients’ survival in basal-like breast cancer patients. Despite ERBB3 expression in basal-\\nlike/triple-negative breast cancer appearing to be low, our study suggests that the NRG1 β/\\nERBB3/ERBB2 axis in basal-like/triple-negative breast cancer patients may support tumor\\ncell dissemination by promoting anchorage-independent cell growth. Importantly, NRG1 β-\\ninduced anchorage-independent growth of basal-like/triple-negative breast cancer cells\\ncould be inhibited by the administration of anti-HER2 agents, in particular, pertuzumab.\\nThe neutralization of the NRG1 β/ERBB3/ERBB2 axis deserves further studies as a thera-\\npeutic strategy for basal-like/triple-negative breast cancers, which today have very limited\\ntreatment opportunities.\\nSupplementary Materials: The following supporting information can be downloaded at: https:\\n//www.mdpi.com/article/10.3390/cancers14071603/s1, Figure S1. NRG1 expression in normal\\nbreast tissue and in breast cancer patients stratiﬁed for clinical and molecular subtypes. (a) Box\\nplot of NRG1 gene expression in normal breast tissues and in breast cancer patients stratiﬁed for\\nclinical subtypes ( n= 833 patients); (b) Box plot of NRG1 gene expression in breast cancer patients\\nstratiﬁed for molecular subtypes (PAM50) ( n= 9672 patients); Figure S2. mRNA expression of ERBB3\\nand NRG1 in breast cell lines. (a–f) mRNAs expression of (a–c) ERBB3 and (d–f) NRG1 in 51 breast\\ncancer cell lines stratiﬁed for (a,b,d,e) molecular clusters and (c,f) clinical subtypes; Figure S3. mRNA\\nexpression of ERBB2 , Estrogen Receptor ( ESR1 ), and Progesterone Receptor ( PR) in breast cell lines.\\n(a–i) mRNAs expression of (a–c) ERBB2 , (d–f) ESR1 and (g–i) PRin breast cancer cell lines stratiﬁed\\nfor (a,c,d,f,g,i) molecular clusters and (b,e,h) clinical subtypes; Figure S4. Time-lapse analysis of\\nbasal-like breast cells treated in vitro with Neuregulin 1 (NRG1). Time-lapse representative images of\\nMCF10A cells treated with/without NRG1 β(10 ng/mL) up to 72 h (1 h, 6 h, 12 h, 24 h, 36 h, 48 h and\\n72 h); Figure S5. Expression of basal/myoepithelial markers in breast cancer patients stratiﬁed for\\nmolecular subtypes. (a–c) mRNA levels of (a) cytokeratin 5 ( n= 9353 patients), (b) cytokeratin 14\\n(n= 9727 patients), and (c) cytokeratin 17 ( n= 9727 patients) in breast cancer patients stratiﬁed for\\nmolecular subtypes (PAM 50); Figure S6. Expression of basal/myoepithelial markers in breast cell\\nlines stratiﬁed for molecular cluster and clinical subtypes. (a–i) mRNAs expression of (a,d,g) Keratin 5\\n(KRT5 ), (b,e,h) Keratin 14 ( KRT14) , (c,f,i) Keratin 17 ( KRT17 ) in breast cancer cell lines stratiﬁed for\\n(a–c,g–i) molecular clusters and (d–f) clinical subtypes; Figure S7. Expression of p63in breast cancer\\npatients and cell lines stratiﬁed for molecular cluster and clinical subtypes. (a) TP63 mRNA levels in\\nbreast cancer patients stratiﬁed for molecular subtypes (PAM 50); (b–d) mRNAs expression of TP63\\nin breast cancer cell lines stratiﬁed for (b, d) molecular clusters and (c) clinical subtypes; Figure S8.\\nAdministration of NRG1 βdoes not affect the survival of basal-like/triple-negative breast cells under\\nanchorage-independent conditions. (a–c) Flow cytometry analysis of the percentage of early (lower\\nright quadrant) and late (upper right quadrant) apoptosis in MCF10A control cells and treated with\\nNRG1 β(10 ng/mL) after (a) 24, (b) 48, and (c) 72 h, respectively. In the ﬂow cytometry proﬁle X-axis\\nand Y-axis exhibit the APC-Annexin V staining and the Propidium Iodide (PI) staining, respectively.\\nGraphs show the percentage of living, early, and apoptotic/necrotic cells at 24, 48, and 72 h; Figure S9 .\\nThe original western blot ﬁgures; Table S1. Sequences of the primers used in this study to analyze\\nmRNA levels by Reat Time PCR.\\nAuthor Contributions: Conceptualization, G.D.; Funding acquisition, M.L. and G.D.; Investigation,\\nC.M., M.L. and G.D.; Methodology, C.M., A.M., F.P ., C.B., F.S., S.D.P ., D.R., R.T., M.S. and E.P .; Project\\nadministration, G.D.; Supervision, C.V ., M.L. and G.D.; Writing—original draft, C.M., M.L. and G.D.\\nAll authors have read and agreed to the published version of the manuscript.Cancers 2022 ,14, 1603 17 of 19\\nFunding: This project was supported by AIRC to Gabriele D’Uva (Grant Number: MFAG 24684), by\\nFondazione Cariplo to Gabriele D’Uva and Mattia Lauriola (Grant Number: GR 2017-0800), and by\\nthe Ministry of Health—Ricerca Corrente—IRCCS MultiMedica.\\nData Availability Statement: The data presented in this study are available in the Supple-\\nmentary Material.\\nConﬂicts of Interest: The authors declare no conﬂict of interest. The funders had no role in the design\\nof the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript, or\\nin the decision to publish the results.\\nReferences\\n1. Yarden, Y.; Pines, G. The ERBB network: At last, cancer therapy meets systems biology. Nat. Rev. Cancer 2012 ,12, 553–563.\\n[CrossRef] [PubMed]\\n2. Arteaga, C.L.; Engelman, J.A. ERBB receptors: From oncogene discovery to basic science to mechanism-based cancer therapeutics.\\nCancer Cell 2014 ,25, 282–303. [CrossRef] [PubMed]\\n3. Yarden, Y .; Sliwkowski, M.X. Untangling the ErbB signalling network. Nat. Rev. Mol. Cell Biol. 2001 ,2, 127–137. [CrossRef] [PubMed]\\n4. Citri, A.; Skaria, K.B.; Yarden, Y. The deaf and the dumb: The biology of ErbB-2 and ErbB-3. Exp Cell Res 2003 ,\\n284, 54–65. [CrossRef]\\n5. Baselga, J.; Swain, S.M. Novel anticancer targets: Revisiting ERBB2 and discovering ERBB3. Nat. Rev. Cancer 2009 ,9, 463–475.\\n[CrossRef] [PubMed]\\n6. Jaiswal, B.S.; Kljavin, N.M.; Stawiski, E.W.; Chan, E.; Parikh, C.; Durinck, S.; Chaudhuri, S.; Pujara, K.; Guillory, J.;\\nEdgar, K.A.; et al. Oncogenic ERBB3 mutations in human cancers. Cancer Cell 2013 ,23, 603–617. [CrossRef] [PubMed]\\n7. Kiavue, N.; Cabel, L.; Melaabi, S.; Bataillon, G.; Callens, C.; Lerebours, F.; Pierga, J.-Y.; Bidard, F.-C. ERBB3 mutations in cancer:\\nBiological aspects, prevalence and therapeutics. Oncogene 2020 ,39, 487–502. [CrossRef]\\n8. Tzahar, E.; Waterman, H.; Chen, X.; Levkowitz, G.; Karunagaran, D.; Lavi, S.; Ratzkin, B.J.; Yarden, Y. A hierarchical network of\\ninterreceptor interactions determines signal transduction by Neu differentiation factor/neuregulin and epidermal growth factor.\\nMol. Cell. Biol. 1996 ,16, 5276–5287. [CrossRef]\\n9. Holbro, T.; Beerli, R.R.; Maurer, F.; Koziczak, M.; Barbas, C.F.; Hynes, N.E. The ErbB2/ErbB3 heterodimer functions as an oncogenic\\nunit: ErbB2 requires ErbB3 to drive breast tumor cell proliferation. Proc. Natl. Acad. Sci. USA 2003 ,100, 8933–8938. [CrossRef]\\n10. Vaught, D.B.; Stanford, J.C.; Young, C.; Hicks, D.J.; Wheeler, F.; Rinehart, C.; Sanchez, V .; Koland, J.; Muller, W.J.;\\nArteaga, C.L.; et al. HER3 Is Required for HER2-Induced Preneoplastic Changes to the Breast Epithelium and Tumor\\nFormation. Cancer Res. 2012 ,72, 2672–2682. [CrossRef] [PubMed]\\n11. Lee, C.Y.-F.; Lin, Y.; Bratman, S.V .; Feng, W.; Kuo, A.H.; Scheeren, F.A.; Engreitz, J.M.; Varma, S.; West, R.B.; Diehn, M. Neuregulin\\nautocrine signaling promotes self-renewal of breast tumor-initiating cells by triggering HER2/HER3 activation. Cancer Res. 2014 ,\\n74, 341–352. [CrossRef]\\n12. Jeong, H.; Kim, J.; Lee, Y.; Seo, J.H.; Hong, S.R.; Kim, A. Neuregulin-1 induces cancer stem cell characteristics in breast cancer cell\\nlines. Oncol. Rep. 2014 ,32, 1218–1224. [CrossRef] [PubMed]\\n13. Harbeck, N.; Penault-Llorca, F.; Cortes, J.; Gnant, M.; Houssami, N.; Poortmans, P .; Ruddy, K.; Tsang, J.; Cardoso, F. Breast cancer.\\nNat. Rev. Dis. Prim. 2019 ,5, 66. [CrossRef] [PubMed]\\n14. Loibl, S.; Poortmans, P .; Morrow, M.; Denkert, C.; Curigliano, G. Breast cancer. Lancet 2021 ,397, 1750–1769. [CrossRef]\\n15. Núñez Abad, M.; Calabuig-Fariñas, S.; Lobo de Mena, M.; Torres-Mart ínez, S.; Garc ía Gonz ález, C.; Garc ía Garc ía, J.Á.;\\nIranzo Gonz ález-Cruz, V . ; Camps Herrero, C. Programmed Death-Ligand 1 (PD-L1) as Immunotherapy Biomarker in Breast\\nCancer. Cancers 2022 ,14, 307. [CrossRef]\\n16. Perou, C.M.; Sorlie, T.; Eisen, M.B.; van de Rijn, M.; Jeffrey, S.S.; Rees, C.A.; Pollack, J.R.; Ross, D.T.; Johnsen, H.; Akslen, L.A.; et al.\\nMolecular portraits of human breast tumours. Nature 2000 ,406, 747–752. [CrossRef] [PubMed]\\n17. Chung, C.H.; Bernard, P .S.; Perou, C.M. Molecular portraits and the family tree of cancer. Nat. Genet. 2002 ,32, 533–540.\\n[CrossRef] [PubMed]\\n18. van ’t Veer, L.J.; Dai, H.; van de Vijver, M.J.; He, Y.D.; Hart, A.A.M.; Mao, M.; Peterse, H.L.; van der Kooy, K.; Marton, M.J.;\\nWitteveen, A.T.; et al. Gene expression proﬁling predicts clinical outcome of breast cancer. Nature 2002 ,415, 530–536.\\n[CrossRef] [PubMed]\\n19. Schnitt, S.J. Classiﬁcation and prognosis of invasive breast cancer: From morphology to molecular taxonomy. Mod. Pathol. 2010 ,\\n23(Suppl. 2), S60–S64. [CrossRef] [PubMed]\\n20. Nielsen, T.O.; Hsu, F.D.; Jensen, K.; Cheang, M.; Karaca, G.; Hu, Z.; Hernandez-Boussard, T.; Livasy, C.; Cowan, D.;\\nDressler, L.; et al. Immunohistochemical and clinical characterization of the basal-like subtype of invasive breast carcinoma. Clin.\\nCancer Res. 2004 ,10, 5367–5374. [CrossRef] [PubMed]\\n21. Cheang, M.C.U.; Voduc, D.; Bajdik, C.; Leung, S.; McKinney, S.; Chia, S.K.; Perou, C.M.; Nielsen, T.O. Basal-like breast cancer\\ndeﬁned by ﬁve biomarkers has superior prognostic value than triple-negative phenotype. Clin. Cancer Res. 2008 ,14, 1368–1376.\\n[CrossRef] [PubMed]Cancers 2022 ,14, 1603 18 of 19\\n22. Bertucci, F.; Finetti, P .; Cervera, N.; Esterni, B.; Hermitte, F.; Viens, P .; Birnbaum, D. How basal are triple-negative breast cancers?\\nInt. J. Cancer 2008 ,123, 236–240. [CrossRef] [PubMed]\\n23. Marra, A.; Trapani, D.; Viale, G.; Criscitiello, C.; Curigliano, G. Practical classiﬁcation of triple-negative breast cancer: Intratumoral\\nheterogeneity, mechanisms of drug resistance, and novel therapies. NPJ Breast Cancer 2020 ,6, 54. [CrossRef]\\n24. Badve, S.; Dabbs, D.J.; Schnitt, S.J.; Baehner, F.L.; Decker, T.; Eusebi, V .; Fox, S.B.; Ichihara, S.; Jacquemier, J.; Lakhani, S.R.; et al.\\nBasal-like and triple-negative breast cancers: A critical review with an emphasis on the implications for pathologists and\\noncologists. Mod. Pathol. 2011 ,24, 157–167. [CrossRef] [PubMed]\\n25. Yin, L.; Duan, J.-J.; Bian, X.-W.; Yu, S.-C. Triple-negative breast cancer molecular subtyping and treatment progress. Breast Cancer\\nRes. 2020 ,22, 61. [CrossRef]\\n26. Jézéquel, P .; Gouraud, W.; Ben Azzouz, F.; Gu érin-Charbonnel, C.; Juin, P .P .; Lasla, H.; Campone, M. bc-GenExMiner 4.5: New\\nmining module computes breast cancer differential gene expression analyses. Database 2021 ,2021 , baab007. [CrossRef] [PubMed]\\n27. Chandrashekar, D.S.; Bashel, B.; Balasubramanya, S.A.H.; Creighton, C.J.; Ponce-Rodriguez, I.; Chakravarthi, B.V .S.K.;\\nVarambally, S. UALCAN: A Portal for Facilitating Tumor Subgroup Gene Expression and Survival Analyses. Neoplasia 2017 ,\\n19, 649–658. [CrossRef]\\n28. Ringn ér, M.; Fredlund, E.; Häkkinen, J.; Borg, Å.; Staaf, J. GOBO: Gene expression-based outcome for breast cancer online. PLoS\\nONE 2011 ,6, e17911. [CrossRef] [PubMed]\\n29. Neve, R.M.; Chin, K.; Fridlyand, J.; Yeh, J.; Baehner, F.L.; Fevr, T.; Clark, L.; Bayani, N.; Coppe, J.P .; Tong, F.; et al. A collection of\\nbreast cancer cell lines for the study of functionally distinct cancer subtypes. Cancer Cell 2006 ,10, 515–527. [CrossRef] [PubMed]\\n30. Györffy, B.; Lanczky, A.; Eklund, A.C.; Denkert, C.; Budczies, J.; Li, Q.; Szallasi, Z. An online survival analysis tool to rapidly\\nassess the effect of 22,277 genes on breast cancer prognosis using microarray data of 1809 patients. Breast Cancer Res. Treat. 2010 ,\\n123, 725–731. [CrossRef]\\n31. Pradeep, C.R.; Kostler, W.J.; Lauriola, M.; Granit, R.Z.; Zhang, F.; Jacob-Hirsch, J.; Rechavi, G.; Nair, H.B.; Hennessy, B.T.;\\nGonzalez-Angulo, A.M.; et al. Modeling ductal carcinoma in situ: A HER2-Notch3 collaboration enables luminal ﬁlling. Oncogene\\n2012 ,31, 907–917. [CrossRef] [PubMed]\\n32. Pradeep, C.R.; Zeisel, A.; Kostler, W.J.; Lauriola, M.; Jacob-Hirsch, J.; Haibe-Kains, B.; Amariglio, N.; Ben-Chetrit, N.; Emde, A.;\\nSolomonov, I.; et al. Modeling invasive breast cancer: Growth factors propel progression of HER2-positive premalignant lesions.\\nOncogene 2012 ,31, 3569–3583. [CrossRef]\\n33. Parker, J.S.; Mullins, M.; Cheang, M.C.U.; Leung, S.; Voduc, D.; Vickery, T.; Davies, S.; Fauron, C.; He, X.; Hu, Z.; et al. Supervised\\nrisk predictor of breast cancer based on intrinsic subtypes. J. Clin. Oncol. 2009 ,27, 1160–1167. [CrossRef] [PubMed]\\n34. Qu, Y.; Han, B.; Yu, Y.; Yao, W.; Bose, S.; Karlan, B.Y.; Giuliano, A.E.; Cui, X. Evaluation of MCF10A as a Reliable Model for\\nNormal Human Mammary Epithelial Cells. PLoS ONE 2015 ,10, e0131285. [CrossRef] [PubMed]\\n35. Lauriola, M.; Enuka, Y.; Zeisel, A.; D’Uva, G.; Roth, L.; Sharon-Sevilla, M.; Lindzen, M.; Sharma, K.; Nevo, N.; Feldman, M.; et al.\\nDiurnal suppression of EGFR signalling by glucocorticoids and implications for tumour progression and treatment. Nat. Commun.\\n2014 ,5, 5073. [CrossRef] [PubMed]\\n36. Amit, I.; Citri, A.; Shay, T.; Lu, Y.; Katz, M.; Zhang, F.; Tarcic, G.; Siwak, D.; Lahad, J.; Jacob-Hirsch, J.; et al. A module of negative\\nfeedback regulators deﬁnes growth factor signaling. Nat. Genet. 2007 ,39, 503–512. [CrossRef]\\n37. Muellner, M.K.; Mair, B.; Ibrahim, Y.; Kerzendorfer, C.; Lechtermann, H.; Trefzer, C.; Klepsch, F.; Müller, A.C.; Leitner, E.;\\nMacho-Maschler, S.; et al. Targeting a cell state common to triple-negative breast cancers. Mol. Syst. Biol. 2015 ,11, 789. [CrossRef]\\n38. Chavez, K.J.; Garimella, S.V .; Lipkowitz, S. Triple negative breast cancer cell lines: One tool in the search for better treatment of\\ntriple negative breast cancer. Breast Dis. 2010 ,32, 35–48. [CrossRef]\\n39. Saeki, K.; Chang, G.; Kanaya, N.; Wu, X.; Wang, J.; Bernal, L.; Ha, D.; Neuhausen, S.L.; Chen, S. Mammary cell gene expression\\natlas links epithelial cell remodeling events to breast carcinogenesis. Commun. Biol. 2021 ,4, 660. [CrossRef]\\n40. Wuidart, A.; Sifrim, A.; Fioramonti, M.; Matsumura, S.; Brisebarre, A.; Brown, D.; Centonze, A.; Dannau, A.; Dubois, C.;\\nVan Keymeulen, A.; et al. Early lineage segregation of multipotent embryonic mammary gland progenitors. Nat. Cell Biol. 2018 ,\\n20, 666–676. [CrossRef]\\n41. Cristea, S.; Polyak, K. Dissecting the mammary gland one cell at a time. Nat. Commun. 2018 ,9, 2473. [CrossRef] [PubMed]\\n42. Fu, N.Y.; Nolan, E.; Lindeman, G.J.; Visvader, J.E. Stem Cells and the Differentiation Hierarchy in Mammary Gland Development.\\nPhysiol. Rev. 2020 ,100, 489–523. [CrossRef]\\n43. Seldin, L.; Le Guelte, A.; Macara, I.G. Epithelial plasticity in the mammary gland. Curr. Opin. Cell Biol. 2017 ,49, 59–63.\\n[CrossRef] [PubMed]\\n44. Hennighausen, L.; Robinson, G.W. Information networks in the mammary gland. Nat. Rev. Mol. Cell Biol. 2005 ,6, 715–725. [CrossRef]\\n45. Stingl, J.; Caldas, C. Molecular heterogeneity of breast carcinomas and the cancer stem cell hypothesis. Nat. Rev. Cancer 2007 ,\\n7, 791–799. [CrossRef]\\n46. Sørlie, T.; Perou, C.M.; Tibshirani, R.; Aas, T.; Geisler, S.; Johnsen, H.; Hastie, T.; Eisen, M.B.; van de Rijn, M.; Jeffrey, S.S.; et al.\\nGene expression patterns of breast carcinomas distinguish tumor subclasses with clinical implications. Proc. Natl. Acad. Sci. U SA\\n2001 ,98, 10869–10874. [CrossRef] [PubMed]\\n47. Hynes, N.E.; Lane, H.A. ERBB receptors and cancer: The complexity of targeted inhibitors. Nat. Rev. Cancer 2005 ,5, 341–354. [CrossRef]Cancers 2022 ,14, 1603 19 of 19\\n48. Junttila, T.T.; Akita, R.W.; Parsons, K.; Fields, C.; Lewis Phillips, G.D.; Friedman, L.S.; Sampath, D.; Sliwkowski, M.X. Ligand-\\nindependent HER2/HER3/PI3K complex is disrupted by trastuzumab and is effectively inhibited by the PI3K inhibitor GDC-0941.\\nCancer Cell 2009 ,15, 429–440. [CrossRef] [PubMed]\\n49. Cho, H.-S.; Mason, K.; Ramyar, K.X.; Stanley, A.M.; Gabelli, S.B.; Denney, D.W.; Leahy, D.J. Structure of the extracellular region of\\nHER2 alone and in complex with the Herceptin Fab. Nature 2003 ,421, 756–760. [CrossRef]\\n50. Capelan, M.; Pugliano, L.; De Azambuja, E.; Bozovic, I.; Saini, K.S.; Sotiriou, C.; Loi, S.; Piccart-Gebhart, M.J. Pertuzumab: New\\nhope for patients with HER2-positive breast cancer. Ann. Oncol. 2013 ,24, 273–282. [CrossRef]\\n51. Agus, D.B.; Akita, R.W.; Fox, W.D.; Lewis, G.D.; Higgins, B.; Pisacane, P .I.; Lofgren, J.A.; Tindell, C.; Evans, D.P .; Maiese, K.; et al.\\nTargeting ligand-activated ErbB2 signaling inhibits breast and prostate tumor growth. Cancer Cell 2002 ,2, 127–137. [CrossRef]\\n52. Miglietta, F.; Griguolo, G.; Bottosso, M.; Giarratano, T.; Lo Mele, M.; Fassan, M.; Cacciatore, M.; Genovesi, E.; De Bartolo, D.;\\nVernaci, G.; et al. Evolution of HER2-low expression from primary to recurrent breast cancer. NPJ Breast Cancer 2021 ,7, 137.\\n[CrossRef] [PubMed]\\n53. Chaffer, C.L.; Weinberg, R.A. A perspective on cancer cell metastasis. Science 2011 ,331, 1559–1564. [CrossRef]\\n54. Shu, L.; Chen, A.; Li, L.; Yao, L.; He, Y.; Xu, J.; Gu, W.; Li, Q.; Wang, K.; Zhang, T.; et al. NRG1 regulates Fra-1 transcription\\nand metastasis of triple-negative breast cancer cells via the c-Myc ubiquitination as manipulated by ERK1/2-mediated Fbxw7\\nphosphorylation. Oncogene 2022 ,41, 907–919. [CrossRef]\\n55. Aceto, N.; Duss, S.; MacDonald, G.; Meyer, D.S.; Roloff, T.-C.; Hynes, N.E.; Bentires-Alj, M. Co-expression of HER2 and HER3\\nreceptor tyrosine kinases enhances invasion of breast cells via stimulation of interleukin-8 autocrine secretion. Breast Cancer Res.\\n2012 ,14, R131. [CrossRef] [PubMed]\\n56. Banerji, U.; van Herpen, C.M.L.; Saura, C.; Thistlethwaite, F.; Lord, S.; Moreno, V .; Macpherson, I.R.; Boni, V .; Rolfo, C.;\\nde Vries, E.G.E.; et al. Trastuzumab duocarmazine in locally advanced and metastatic solid tumours and HER2-expressing breast\\ncancer: A phase 1 dose-escalation and dose-expansion study. Lancet. Oncol. 2019 ,20, 1124–1135. [CrossRef]\\n57. Modi, S.; Park, H.; Murthy, R.K.; Iwata, H.; Tamura, K.; Tsurutani, J.; Moreno-Aspitia, A.; Doi, T.; Sagara, Y.; Redfern, C.; et al.\\nAntitumor Activity and Safety of Trastuzumab Deruxtecan in Patients with HER2-Low-Expressing Advanced Breast Cancer:\\nResults from a Phase Ib Study. J. Clin. Oncol. 2020 ,38, 1887–1896. [CrossRef]\\n58. Eiger, D.; Agostinetto, E.; Sa úde-Conde, R.; de Azambuja, E. The Exciting New Field of HER2-Low Breast Cancer Treatment.\\nCancers 2021 ,13, 1015. [CrossRef]\\n59. Gianni, L.; Llad ó, A.; Bianchi, G.; Cortes, J.; Kellokumpu-Lehtinen, P .-L.; Cameron, D.A.; Miles, D.; Salvagni, S.; Wardley, A.;\\nGoeminne, J.-C.; et al. Open-label, phase II, multicenter, randomized study of the efﬁcacy and safety of two dose levels of\\nPertuzumab, a human epidermal growth factor receptor 2 dimerization inhibitor, in patients with human epidermal growth\\nfactor receptor 2-negative metastat. J. Clin. Oncol. 2010 ,28, 1131–1137. [CrossRef]\\n60. Chervo, M.F.; Cordo Russo, R.I.; Petrillo, E.; Izzo, F.; De Martino, M.; Bellora, N.; Cenciarini, M.E.; Chiauzzi, V .A.;\\nSanta Mar ía de la Parra, L. ; Pereyra, M.G.; et al. Canonical ErbB-2 isoform and ErbB-2 variant c located in the nucleus drive triple\\nnegative breast cancer growth. Oncogene 2020 ,39, 6245–6262. [CrossRef]\\n61. Phillips, G.D.L.; Fields, C.T.; Li, G.; Dowbenko, D.; Schaefer, G.; Miller, K.; Andre, F.; Burris, H.A.; Albain, K.S.; Harbeck, N.; et al.\\nDual targeting of HER2-positive cancer with trastuzumab emtansine and pertuzumab: Critical role for neuregulin blockade in\\nantitumor response to combination therapy. Clin. Cancer Res. 2014 ,20, 456–468. [CrossRef] [PubMed]\\n62. Erickson, A.W.; Habbous, S.; Hoey, C.; Jerzak, K.J.; Das, S. Dual- versus single-agent HER2 inhibition and incidence of intracranial\\nmetastatic disease: A systematic review and meta-analysis. NPJ Breast Cancer 2021 ,7, 17. [CrossRef] [PubMed]'"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "def load_pdf():\n",
    "    text=\"\"\n",
    "    loader = PyPDFDirectoryLoader(\".\", glob=\"**/[!.]*.pdf\")\n",
    "    for page in loader.load():\n",
    "       text+=page.page_content\n",
    "    return text\n",
    "load_pdf()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'- Higher ERBB3 mRNA expression is correlated with shorter relapse-free survival in basal-like breast cancers.\\n- Neuregulin 1 beta (NRG1β) does not significantly affect cellular proliferation or basal migratory ability in basal-like breast cells.\\n- NRG1β administration induces 3D cell growth and upregulates ERBB3 and ERBB2 protein abundance.\\n- The anti-HER2 monoclonal antibody pertuzumab prevents NRG1β-induced 3D cell growth.\\n- Similar results were observed in basal-like/triple-negative breast cancer cellular models.\\n- ERBB2 neutralization, particularly by pertuzumab, inhibits the anchorage-independent growth of basal-like breast cancer cells.\\n- The NRG1/ERBB3/ERBB2 pathway promotes the anchorage-independent growth of basal-like breast cancer cells.\\n- The interception of the NRG1/ERBB3/ERBB2 signaling axis could lead to novel anticancer strategies for basal-like breast cancer patients. \\n\\nTargets mentioned: ERBB3, HER2, NRG1β, pertuzumab.'"
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "loader = CSVLoader(file_path='key_points.csv', csv_args={\n",
    "    'delimiter': ',',\n",
    "    'quotechar': '\"',\n",
    "    'fieldnames': ['abstract', 'keypoints', 'target']\n",
    "})\n",
    "documents = loader.load()\n",
    "embeddings = OpenAIEmbeddings()\n",
    "db = Chroma.from_documents(documents, embeddings)\n",
    "\n",
    "def retrieve_info(query):\n",
    "    response = db.similarity_search(query, \n",
    "                                    k=3) #retrieves top 3\n",
    "    contents = [doc.page_content for doc in response]\n",
    "    return contents\n",
    "\n",
    "llm = ChatOpenAI(temperature=0, model=\"gpt-3.5-turbo-16k-0613\")\n",
    "\n",
    "template = \"\"\"\n",
    "You are an expert summarizer on breast cancer research papers. I am going to share with you an academic paper\n",
    "and you will give me a summarization of the most relevant information in 10 bullet points or less, based on past\n",
    "summaries.\n",
    "You will follow all the rules below: \n",
    "\n",
    "1/ Response must be similar to past summaries, in terms of length, and tone of voice\n",
    "2/ You must list all the targets (ADC, HER3, ADC etc.) that the paper has mentioned, after the bullet points are formed\n",
    "3/ If the summaries are found to be irrelevant, you may mention that you are unsure\n",
    "\n",
    "Below is a typical academic paper on cancer: \n",
    "{message}\n",
    "\n",
    "Here is a list of how the summaries are like for a similar paper: \n",
    "{summary}\n",
    "\n",
    "Please write the summary for the paper below:\n",
    "\"\"\"\n",
    "\n",
    "prompt = PromptTemplate(\n",
    "    input_variables=[\"message\",\"summary\"],\n",
    "    template=template\n",
    "\n",
    ")\n",
    "chain = LLMChain(llm=llm, prompt=prompt)\n",
    "\n",
    "def load_pdf():\n",
    "    text=\"\"\n",
    "    loader = PyPDFDirectoryLoader(\".\", glob=\"**/[!.]*.pdf\")\n",
    "    for page in loader.load():\n",
    "       text+=page.page_content\n",
    "    return text\n",
    "\n",
    "def generate_response(message):\n",
    "    summary=retrieve_info(message)\n",
    "    response = chain.run(message=message, summary=summary)\n",
    "    return response\n",
    "\n",
    "message = \"\"\"\n",
    "Abstract: ERBB3, also known as HER3, is a tyrosine kinase transmembrane receptor of the ERBB\n",
    "family. Upon binding to neuregulin 1 (NRG1), ERBB3 preferentially dimerizes with HER2 (ERBB2),\n",
    "in turn inducing aggressive features in several cancer types. The analysis of a dataset of breast\n",
    "cancer patients unveiled that higher ERBB3 mRNA expression correlates with shorter relapse-free\n",
    "survival in basal-like breast cancers, despite low ERBB3 expression in this breast cancer subtype.\n",
    "Administration of neuregulin 1 beta (NRG1β) significantly affected neither cellular proliferation nor\n",
    "the basal migratory ability of basal-like/triple-negative quasi-normal MCF10A breast cells, cultured\n",
    "in mono-layer conditions. Furthermore, no significant regulation in cell morphology or in the\n",
    "expression of basal/myoepithelial and luminal markers was observed upon stimulation with NRG1β.\n",
    "In non-adherent conditions, NRG1βadministration to MCF10A cells did not significantly influence\n",
    "cell survival; however, it robustly induced cell growth as spheroids (3D growth). Intriguingly, a\n",
    "remarkable upregulation of ERBB3 and ERBB2 protein abundance was observed in 3D compared\n",
    "to 2D cell cultures, and NRG1β-induced 3D cell growth was efficiently prevented by the anti-\n",
    "HER2 monoclonal antibody pertuzumab. Similar results were obtained by the analysis of basal-\n",
    "like/triple-negative breast cancer cellular models, MDA-MB-468 and MDA-MB-231 cells, in which\n",
    "NRG1βinduced anchorage-independent cell growth that in turn was prevented or reduced by the\n",
    "simultaneous administration of anti-HER2 neutralizing antibodies. Finally, the ability of pertuzumab\n",
    "in suppressing NRG1β-induced 3D growth was also evaluated and confirmed in MCF10A engineered\n",
    "with HER2-overexpression. We suggest that the NRG1/ERBB3/ERBB2 pathway promotes the\n",
    "anchorage-independent growth of basal-like breast cancer cells. Importantly, we provide evidence\n",
    "that ERBB2 neutralization, in particular by pertuzumab, robustly inhibits this process. Our results\n",
    "pave the way towards the development of novel anticancer strategies for basal-like breast cancer\n",
    "patients based on the interception of the NRG1/ERBB3/ERBB2 signaling axis\"\"\"\n",
    "generate_response(message) "
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "py310",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.0"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
